<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:dc="http://purl.org/dc/terms/" xmlns:doi="http://dx.doi.org/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#" xml:lang="en">

<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">


<meta name="wpd_version" content="0.2">
<meta name="wpd_baseurl" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455">
<meta name="wpd_url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455">
<meta name="wpd_date" content="2011-02-13T11:16Z">






<script type="text/javascript">var _sf_startpt=(new Date()).getTime()</script>
   <title>PLoS ONE: Drop-on-Demand Single Cell Isolation and Total RNA Analysis</title>




<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7">

<link rel="shortcut icon" href="about:blank?favicon.ico" type="image/x-icon">
<link rel="home" title="home" href="http://www.plosone.org/home.action">


<link rel="alternate" type="application/atom+xml" title="PLoS ONE: New Articles" href="http://www.plosone.org/article/feed">


<!-- global_css.ftl -->








<meta name="description" content="PLoS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world.">

<meta name="keywords" content="PLoS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering">


<!--
<rdf:RDF xmlns="http://web.resource.org/cc/"
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
<Work rdf:about="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0017455?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0017455">
   <license rdf:resource="http://creativecommons.org/licenses/by/2.5/" />
</Work>
<License rdf:about="http://creativecommons.org/licenses/by/2.5/">
   <permits rdf:resource="http://web.resource.org/cc/Reproduction" />
   <permits rdf:resource="http://web.resource.org/cc/Distribution" />
   <requires rdf:resource="http://web.resource.org/cc/Notice" />
   <requires rdf:resource="http://web.resource.org/cc/Attribution" />
   <permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" />
</License>
<rdf:Description
     rdf:about="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0017455?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0017455"
     dc:identifier="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0017455?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0017455"
     dc:title="PLoS ONE: Drop-on-Demand Single Cell Isolation and Total RNA Analysis"
       trackback:ping="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455/trackback"
     />
</rdf:RDF>

-->

<link rel="stylesheet" type="text/css" href="infodoi10.1371journal.pone.css" media="all">
</head>
<body>
<span property="dc:language" content="en"></span>
<!-- begin : container -->
<div id="container">
  <!-- begin : top banner external ad space -->
  <div id="topBanner">


<!-- begin : left banner slot -->
<div class="left">
  <iframe id="aa97ff20" name="aa97ff20" src="infodoi10.1371journal.pone_1.html" frameborder="0" height="60" scrolling="no" width="468">&lt;a
    href='http://ads.plos.org/www/delivery/ck.php?n=aa97ff20&amp;cb=1089'
    target='_top'&gt;&lt;img src='http://ads.plos.org/www/delivery/avw.php?zoneid=287&amp;cb=5965&amp;n=aa97ff20'
    border='0' alt='' /&gt;&lt;/a&gt;</iframe>
</div>
<!-- end : left banner slot -->
<!-- begin : right banner slot -->
<div class="right">
  <iframe id="acfd0f5a" name="acfd0f5a" src="infodoi10.1371journal.pone_2.html" frameborder="0" height="60" scrolling="no" width="468">&lt;a
    href='http://ads.plos.org/www/delivery/ck.php?n=acfd0f5a&amp;cb=9235'
    target='_top'&gt;&lt;img src='http://ads.plos.org/www/delivery/avw.php?zoneid=40&amp;cb=6107&amp;n=acfd0f5a'
    border='0' alt='' /&gt;&lt;/a&gt;</iframe>
</div>
<!-- end : right banner slot -->
  </div>
  <!-- end : top banner external ad space -->

  <!-- begin : header -->
  <div id="hdr" class="login">

  <!-- begin : logo -->
  <div id="logo" title="PLoS ONE"><a href="http://www.plosone.org/home.action" title="PLoS ONE"><span>PLoS ONE</span></a></div>
  <!-- end : logo -->
  <!-- begin : user controls -->





  <div id="user">
    <div>
      <ul>
        <li><a href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2Finfo%3Adoi%2F10.1371%2Fjournal.pone.0017455" class="feedback"><strong>Login</strong></a> | </li>
        <li><a href="https://register.plos.org/ambra-registration/register.action">Create Account</a> | </li>
        <li class="feedback"><a href="http://www.plosone.org/feedbackCreate.action?page=%252Farticle%252Finfo%253Adoi%252F10.1371%252Fjournal.pone.0017455" title="Send us your feedback">Feedback</a></li>
      </ul>
    </div>
  </div>


  <!-- end : user controls -->
  <!-- begin search links -->
  <ul id="links"><li class="browse"><a href="http://www.plosone.org/article/browse.action?field=date" title="Browse Articles">Browse</a></li><li class="rss"><a href="http://www.plosone.org/static/rssFeeds.action" title="RSS Feeds">RSS</a></li></ul>
  <!-- end : search links -->
  <!-- begin : dashboard -->
  <div id="db">
    <form name="searchForm" action="http://www.plosone.org/search/simpleSearch.action" method="get">
      <input name="from" value="globalSimpleSearch" type="hidden">
      <input name="filterJournals" value="PLoSONE" type="hidden">
      <fieldset>
        <legend>Search</legend>
        <label for="search">Search</label>
        <div class="wrap"><input id="search" name="query" value="Search articles..." onfocus="if(this.value=='Search articles...')value='';" onblur="if(this.value=='')value='Search articles...';" class="searchField" alt="Search articles..." type="text"></div>
        <input src="search_btn1.gif" onclick="submit();" value="ftsearch" alt="SEARCH" tabindex="3" class="button" type="image">
      </fieldset>
    </form>
    <form name="gasf" action="http://www.plosone.org/search/advancedSearch.action" method="get">
<input name="noSearchFlag" value="true" id="noSearchFlag" type="hidden"><input name="query" value="" id="query" type="hidden">    </form>
    <a id="advSearch" href="#" onclick="if(document.searchForm.query.value!='Search articles...')document.gasf.query.value=document.searchForm.query.value;document.gasf.submit();return false;">Advanced Search</a>
  </div>
  <!-- end : dashboard -->
  <!-- begin : navigation -->
  <ul id="nav">
    <li><a href="http://www.plosone.org/home.action" tabindex="101">Home</a></li>
    <li><a href="http://www.plosone.org/static/browse.action" tabindex="102">Browse Articles</a>
        <ul>
           <li><a href="http://www.plosone.org/article/browse.action?field=date">By Publication Date</a></li>
          <li><a href="http://www.plosone.org/article/browse.action">By Subject</a></li>
         <li><a href="http://www.ploscollections.org/static/poneCollections.action">Collections</a></li>
       </ul>
    </li>
    <li><a href="http://www.plosone.org/static/about.action" tabindex="103">About</a>
        <ul>
          <li><a href="http://www.plosone.org/static/information.action">Journal Information</a></li>
          <li><a href="http://www.plosone.org/static/edboard.action">Editorial Board</a></li>
          <li><a href="http://www.plosone.org/static/almInfo.action">Article-Level Metrics</a></li>
          <li><a href="http://www.plosone.org/static/license.action">Open-Access License</a></li>
          <li><a href="http://www.plosone.org/static/contact.action">Contact Us</a></li>
        </ul>
      </li>
    <li><a href="http://www.plosone.org/static/users.action" tabindex="104">For Readers</a>
        <ul>
          <li><a href="http://www.plosone.org/static/commentGuidelines.action">Guidelines for Notes, Comments, and Corrections</a></li>
          <li><a href="http://www.plosone.org/static/ratingGuidelines.action">Guidelines for Rating</a></li>
          <li><a href="http://www.plosone.org/static/help.action">Help Using this Site</a></li>
		  <li><a href="http://www.plosone.org/static/journalClub.action">Journal Club Archives</a></li>
          <li><a href="http://www.plosone.org/static/sitemap.action">Site Map</a></li>
        </ul>
      </li>
    <li><a href="http://www.plosone.org/static/authors.action" tabindex="105">For Authors and Reviewers</a>
        <ul>
          <li><a href="http://www.plosone.org/static/whypublish.action">Why Publish With Us?</a></li>
          <li><a href="http://www.plosone.org/static/review.action">Editorial and Peer-Review Process</a></li>
          <li><a href="http://www.plosone.org/static/policies.action">Editorial and Publishing Policies</a></li>
          <li><a href="http://www.plosone.org/static/guidelines.action">Author Guidelines</a></li>
          <li><a href="http://www.plosone.org/static/figureGuidelines.action">Figure and Table Guidelines</a></li>
          <li><a href="http://www.plosone.org/static/checklist.action">Submit Your Paper</a></li>
          <li><a href="http://www.plosone.org/static/reviewerGuidelines.action">Reviewer Guidelines</a></li>
	  <li><a href="http://www.plosone.org/static/peerReviewers.action">Thanking All Peer Reviewers</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://www.plos.org/" title="Public Library of Science" tabindex="110" class="drop">PLoS.org</a>
        <ul>
          <li><a href="http://www.plos.org/oa/index.html" title="Open Access Statement">Open Access</a></li>
          <li><a href="http://www.plos.org/support/donate.php" title="Join PLoS: Show Your Support">Join PLoS</a></li>
          <li><a href="http://blogs.plos.org/" title="PLoS Blogs">PLoS Blogs</a></li>
 	  <li><a href="http://www.zazzle.com/plos?rf=238434112415374154" title="PLoS Store">PLoS Store</a></li>
	  <li><a href="http://currents.plos.org/" title="PLoS Currents">PLoS Currents</a></li>
          <li><a href="http://www.plos.org/connect.html" title="PLoS.org | Stay Connected">Stay Connected</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://hubs.plos.org/" tabindex="109">Hubs</a>
        <ul>
          <li><a href="http://hubs.plos.org/biodiversity" title="PLoS Hubs: Biodiversity">Biodiversity</a></li>
          <li><a href="http://clinicaltrials.ploshubs.org/" title="PLoS Hub for Clinical Trials">Clinical Trials</a></li>
        </ul>
    </li>
      <li class="journalnav"><a href="http://www.plosjournals.org/" tabindex="108">Journals</a>
        <ul>
	<li><a href="http://www.plosbiology.org/" title="PLoSBiology.org">PLoS Biology</a></li>
	<li><a href="http://www.plosmedicine.org/" title="PLoSMedicine.org">PLoS Medicine</a></li>
	<li><a href="http://www.ploscompbiol.org/" title="PLoSCompBiol.org">PLoS Computational Biology</a></li>
	<li><a href="http://www.plosgenetics.org/" title="PLoSGenetics.org">PLoS Genetics</a></li>
	<li><a href="http://www.plospathogens.org/" title="PLoSPathogens.org">PLoS Pathogens</a></li>
        <li><a href="http://www.plosone.org/" title="PLoSONE.org">PLoS ONE</a></li>
        <li><a href="http://www.plosntds.org/" title="PLoSNTDs.org">PLoS Neglected Tropical Diseases</a></li>
        </ul>
      </li>
    </ul>
  <!-- end : navigation -->
  </div>
  <!-- end : header --> 



<!-- begin : main content -->
<div id="content" class="article" style="visibility: visible;">
<!-- begin : right hand column -->

<div id="rhc" xpathlocation="noDialog">

  <div id="download" class="rhcBox_type1">
    <div class="wrap">
      <ul>
        <li class="download icon"><strong>Download:</strong>
          <a href="http://www.plosone.org/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0017455&representation=PDF" title="Download article PDF">PDF</a> |
          <a href="http://www.plosone.org/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0017455" title="Download citations">Citation</a> |
          <a href="http://www.plosone.org/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0017455&representation=XML" title="Download Article XML">XML</a>
        </li>
        <li class="print icon"><a href="#" onclick="window.print();return false;" title="Print Article"><strong>Print article</strong></a></li>
          <li class="reprint icon"><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0017455&volume=&issue=&title=Drop-on-Demand%20Single%20Cell%20Isolation%20and%20Total%20RNA%20Analysis&author_name=Sangjun%20Moon%2C%20Yun-Gon%20Kim%2C%20Lingsheng%20Dong%2C%20Michael%20Lombardi%2C%20Edward%20Haeggstrom%2C%20Roderick%20V.%20Jensen%2C%20Li-Li%20Hsiao%2C%20Utkan%20Demirci&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a> New & improved!</li>
      </ul>
    </div>
  </div>

  <div id="impact" class="rhcBox_type2">
    <h6>Metrics <a href="http://www.plosone.org/static/help.action#articleMetrics" class="replaced" id="info" title="More information"><span>info</span></a></h6>

    <div id="totalDataInRHC"></div>

    <div id="relatedCitesInRHC"></div>

    <div id="ratingRhc1">

            <h6>Average Rating <a href="http://www.plosone.org/rate/getArticleRatings.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0017455" class="rating">(0 User Ratings)</a></h6>
            <ol>
              <li>
                <ul class="star-rating rating inline-rating-rhc" title="overall">
                    <li class="current-rating overall-rating pct0">Currently 0/5 Stars.</li>
                </ul>
                <a href="javascript:void(0);" onclick="return ambra.domUtil.swapDisplayTextMode(this, 'ratingAverages', null, 'Hide all categories', 'See all categories');" class="rating catAvg">See all categories</a>
                <div id="ratingAverages">
                  <ol class="ratingaverageRatings.">
                    <li><label for="insight">Insight</label>
                      <ul class="star-rating rating" title="insight">
                        <li class="current-rating average pct0">Currently 0/5 Stars.</li>
                      </ul>
                    </li>
                    <li><label for="reliability">Reliability</label>
                      <ul class="star-rating rating" title="reliability">
                        <li class="current-rating average pct0">Currently 0/5 Stars.</li>
                      </ul>
                    </li>
                    <li><label for="style">Style</label>
                      <ul class="star-rating rating" title="style">
                        <li class="current-rating average pct0">Currently 0/5 Stars.</li>
                      </ul>
                    </li>
                  </ol>
                </div>
                <a href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2Finfo%3Adoi%2F10.1371%2Fjournal.pone.0017455" class="rating">Rate This Article</a>
              </li>
            </ol>
            
    </div>
    <div class="more clearfix"><a href="http://www.plosone.org/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0017455" title="More information">More</a></div>
  </div>

  <div id="related" class="rhcBox_type2">
    <h6>Related Content</h6>
    <dl class="related inlineList">
      <dt>Related Subject Categories</dt>
      <dd><a href="http://www.plosone.org/article/browse.action?catName=Cell+Biology" title="Browse the Cell Biology Category">Cell Biology</a></dd>, 
      <dd><a href="http://www.plosone.org/article/browse.action?catName=Genetics+and+Genomics" title="Browse the Genetics and Genomics Category">Genetics and Genomics</a></dd>, 
      <dd><a href="http://www.plosone.org/article/browse.action?catName=Biotechnology" title="Browse the Biotechnology Category">Biotechnology</a></dd>
    </dl>

    <dl class="related">
      <dt>Related Articles on the Web</dt>
      <dd><a href="http://scholar.google.com/scholar?hl=en&lr=&q=related:http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0017455&btnG=Search">Google Scholar</a></dd>
      <dd id="pubMedRelatedLIInRHC" style="display: none;"><a id="pubMedRelatedURLInRHC">PubMed</a></dd>
    </dl>

    <div class="more clearfix"><a href="http://www.plosone.org/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0017455" title="More information">More</a></div>
  </div>

  <div id="share" class="rhcBox_type2">
    <h6>Share this Article <a href="http://www.plosone.org/static/help.action#socialBookmarkLinks" class="replaced" id="info" title="More information"><span>info</span></a></h6>
    <ul>
      <li class="bookmarklets">

          <a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0017455" target="_new"> <img src="16x16_su_solid.gif" alt="StumbleUpon" title="Add to StumbleUpon" border="0"></a>
          <script>function fbs_click() {u='http://dx.plos.org/10.1371/journal.pone.0017455';t='Drop-on-Demand%20Single%20Cell%20Isolation%20and%20Total%20RNA%20Analysis';window.open('http://www.facebook.com/sharer.php?u='+encodeURIComponent(u)+'&t='+encodeURIComponent(t),'sharer','toolbar=0,status=0,width=626,height=436');return false;}</script><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0017455" onclick="return fbs_click()"><img src="facebook_share_icon.gif" alt="Facebook" title="Add to Facebook"></a>       <!-- for mor info, see http://www.facebook.com/share_partners.php -->
          <script type="text/javascript">

            function bookmark_in_connotea(u) {
                a=false; x=window; e=x.encodeURIComponent; d=document;
                w=open('http://www.connotea.org/addpopup?continue=confirm&uri='+e(u),
                    'add', 'width=600, height=400, scrollbars, resizable');
                void(x.setTimeout('w.focus()',200));
            }
          </script>
          <a style="cursor: pointer;" onclick='javascript:bookmark_in_connotea("http://dx.plos.org/10.1371/journal.pone.0017455");'><img src="icon_connotea_16x16.gif" alt="Connotea" title="Add to Connotea"></a>
          <a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0017455&title=Drop-on-Demand%20Single%20Cell%20Isolation%20and%20Total%20RNA%20Analysis" target="_new"><img src="icon_citeulike_16x16.gif" alt="CiteULike" title="Add to CiteULike"></a>
          <a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0017455" title="Add to Mendeley" target="_new"><img src="icon_mendeley_16x16.gif" alt="Bibliography"></a>
          <script type="text/javascript">

            var twtTitle  = 'Drop-on-Demand Single Cell Isolation and Total RNA Analysis';
            var twtUrl    = 'http://dx.plos.org/10.1371/journal.pone.0017455';
            var twtTag    = '#PLoS:'
            var maxLength = 140 - (twtUrl.length + 1);
            if (twtTitle.length > maxLength) {
            twtTitle = twtTitle.substr(0, (maxLength - 10))+'...';  //truncates title to include space for twtTag and ellipsis
            }
            var twtLink = 'http://twitter.com/home?status='+encodeURIComponent(twtTag + ' ' + twtTitle + ' ' + twtUrl);
            document.write('<a href="'+twtLink+'" target="_blank"'+'><img src="/images/icon_twitter.gif"  border="0" alt="Twitter icon" title="Tweet This!" /'+'><'+'/a>');
          </script>
          <script type="text/javascript">

          digg_url = 'http://dx.plos.org/10.1371/journal.pone.0017455';
          digg_skin = 'icon';
          digg_title = 'Drop-on-Demand Single Cell Isolation and Total RNA Analysis';
          digg_bodytext = '';
          digg_topic = '';
          digg_media = 'news';
          digg_window = 'new';

          </script>
          <script src="diggthis.js" type="text/javascript"></script>
      </li>
      <li class="email icon"><a href="http://www.plosone.org/article/emailArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0017455" title="Email this article">Email this article</a></li>
    </ul>
  </div>
</div>
<!-- end : right hand column -->
<form name="articleInfo" id="articleInfo" method="" action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455">
<input name="isAuthor" value="true" type="hidden">
<input name="authorIdList" value="" type="hidden">
<input name="userIdList" value="" type="hidden">
<input name="otherIdList" value="" type="hidden">
<input name="annotationId" value="" type="hidden">
<input name="isResearchArticle" value="true" type="hidden">
</form>
<div id="articleContainer"><span about="http://www.plos.org" typeof="foaf:Organization" property="foaf:name" style="display: none; visibility: hidden;">Public Library of Science</span>
<div id="researchArticle" class="content">
  <span rel="dc:publisher" href="http://www.plos.org"></span>
  <a id="top" name="top"></a>
<div id="contentHeader" xpathlocation="noSelect">
  <p>Open Access</p><p id="articleType">Research Article</p>
</div>

  <h1 xpathlocation="noSelect" property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">Drop-on-Demand Single Cell Isolation and Total RNA Analysis</h1>
  <div property="dc:description" datatype="" style="display: none; visibility: hidden;">
        <p>Technologies that rapidly isolate viable single cells from heterogeneous solutions have significantly contributed to the field of medical genomics. Challenges remain both to enable efficient extraction, isolation and patterning of single cells from heterogeneous solutions as well as to keep them alive during the process due to a limited degree of control over single cell manipulation. Here, we present a microdroplet based method to isolate and pattern single cells from heterogeneous cell suspensions (10% target cell mixture), preserve viability of the extracted cells (97.0&plusmn;0.8%), and obtain genomic information from isolated cells compared to the non-patterned controls. The cell encapsulation process is both experimentally and theoretically analyzed. Using the isolated cells, we identified 11 stem cell markers among 1000 genes and compare to the controls. This automated platform enabling high-throughput cell manipulation for subsequent genomic analysis employs fewer handling steps compared to existing methods.</p>
      </div>
  <span property="dc:date" content="2011-03-11" datatype="xsd:date" rel="dc:identifier" href="http://dx.doi.org/10.1371/journal.pone.0017455"></span>
  <span property="dc:subject" content="Cell Biology"></span>
  <span property="dc:subject" content="Genetics and Genomics"></span>
  <span property="dc:subject" content="Biotechnology"></span>
<form action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455">
  <input name="journalDisplayName" id="journalDisplayName" value="PLoS ONE" type="hidden">
  <input name="crossRefPageURL" id="crossRefPageURL" value="/article/crossref/info%3Adoi%2F10.1371%2Fjournal.pone.0017455" type="hidden">
  <input name="metricsTabURL" id="metricsTabURL" value="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0017455" type="hidden">
  <input name="doi" id="doi" value="info:doi/10.1371/journal.pone.0017455" type="hidden">
  <input name="articleTitleUnformatted" id="articleTitleUnformatted" value="Drop-on-Demand%20Single%20Cell%20Isolation%20and%20Total%20RNA%20Analysis" type="hidden">
  <input name="articlePubDate" id="articlePubDate" value="1299830400000" type="hidden">
</form>
<div class="horizontalTabs" xpathlocation="noSelect">
  <ul id="tabsContainer">
        <li id="article" class="active"><a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455" class="tab" title="Article">Article</a></li>
        <li id="metrics"><a href="http://www.plosone.org/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0017455" class="tab" title="Metrics">Metrics</a></li>
        <li id="related"><a href="http://www.plosone.org/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0017455" class="tab" title="Related Content">Related Content</a></li>
        <li id="comments"><a href="http://www.plosone.org/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0017455" class="tab" title="Comments">Comments: 0</a></li>
  </ul>
</div>
  <div id="articleMenu" xpathlocation="noSelect"> 
    <div class="wrap"> 
      <ul> 
        <li class="annotation icon">To <strong>add a note</strong>, highlight some text. <a href="#" onclick="toggleAnnotation(this, 'public'); return false;" title="Click to turn notes on/off">Hide notes</a></li> 
        <li class="discuss icon"> 
            <a href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2Finfo%3Adoi%2F10.1371%2Fjournal.pone.0017455">Make a general comment</a> 
        </li> 
      </ul>
      <div id="sectionNavTopBox" style="display: none;">
        <p><strong>Jump to</strong></p> 
        <div id="sectionNavTop" class="tools"></div>
      </div>
    </div>
  </div>





<p class="authors" xpathlocation="noSelect"><span rel="dc:creator"><span property="foaf:name">Sangjun Moon</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span property="foaf:name">Yun-Gon Kim</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span property="foaf:name">Lingsheng Dong</span></span><sup><a href="#aff2">2</a></sup>, <span rel="dc:creator"><span property="foaf:name">Michael Lombardi</span></span><sup><a href="#aff2">2</a></sup>, <span rel="dc:creator"><span property="foaf:name">Edward Haeggstrom</span></span><sup><a href="#aff3">3</a></sup>, <span rel="dc:creator"><span property="foaf:name">Roderick V. Jensen</span></span><sup><a href="#aff4">4</a></sup>, <span rel="dc:creator"><span property="foaf:name">Li-Li Hsiao</span></span><sup><a href="#aff2">2</a></sup><sup><a href="#cor1" class="fnoteref">*</a></sup>, <span rel="dc:creator"><span property="foaf:name">Utkan Demirci</span></span><sup><a href="#aff1">1</a></sup><sup>,</sup><sup><a href="#aff2">2</a></sup><sup>,</sup><sup><a href="#aff5">5</a></sup><sup><a href="#cor1" class="fnoteref">*</a></sup></p><p class="affiliations" xpathlocation="noSelect"><a name="aff1" id="aff1"></a><strong>1</strong> Demirci Bio-Acoustic-MEMS in Medicine (BAMM) Laboratory, Center for Bioengineering, Brigham and Women's Hospital, Harvard Medical School, Boston, Massachusetts, United States of America, <a name="aff2" id="aff2"></a><strong>2</strong> Renal Division, Brigham and Women's Hospital, Harvard Medical School, Boston, Massachusetts, United States of America, <a name="aff3" id="aff3"></a><strong>3</strong> Electronics Research Laboratory, Department of Physics, University of Helsinki, Helsinki, Finland, <a name="aff4" id="aff4"></a><strong>4</strong> Department of Biological Sciences, Virginia Tech, Blacksburg, Virginia, United States of America, <a name="aff5" id="aff5"></a><strong>5</strong> Harvard-MIT Division of Health Sciences and Technology, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America</p><div class="abstract" xpathlocation="/article[1]/front[1]/article-meta[1]/abstract[1]"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2 xpathlocation="noSelect">Abstract&nbsp;<a href="#top">Top</a></h2><p xpathlocation="/article[1]/front[1]/article-meta[1]/abstract[1]/p[1]">Technologies that rapidly isolate viable single cells from heterogeneous solutions have significantly contributed to the field of medical genomics. Challenges remain both to enable efficient extraction, isolation and patterning of single cells from heterogeneous solutions as well as to keep them alive during the process due to a limited degree of control over single cell manipulation. Here, we present a microdroplet based method to isolate and pattern single cells from heterogeneous cell suspensions (10% target cell mixture), preserve viability of the extracted cells (97.0&plusmn;0.8%), and obtain genomic information from isolated cells compared to the non-patterned controls. The cell encapsulation process is both experimentally and theoretically analyzed. Using the isolated cells, we identified 11 stem cell markers among 1000 genes and compare to the controls. This automated platform enabling high-throughput cell manipulation for subsequent genomic analysis employs fewer handling steps compared to existing methods.</p>
</div>


<div class="articleinfo" xpathlocation="noSelect"><p><strong>Citation: </strong>Moon S, Kim Y-G, Dong L, Lombardi M, Haeggstrom E, et al.  (2011) Drop-on-Demand Single Cell Isolation and Total RNA Analysis. PLoS ONE 6(3):
          e17455.
            doi:10.1371/journal.pone.0017455</p><p><strong>Editor: </strong>Dimas Covas, University of Sao Paulo - USP, Brazil</p><p></p><p><strong>Received:</strong> September 13, 2010; <strong>Accepted:</strong> February 3, 2011; <strong>Published:</strong> March 11, 2011</p><p><strong>Copyright:</strong> &copy; 2011 Moon et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding:</strong> The authors would like to acknowledge NIH R21 EB007707, and the W.H. Coulter Foundation Young Investigation Award. This was also partially supported by RO1 A1081534, R21 AI087107, and Integration of Medicine and Innovative Technology (CIMIT) under U.S. Army Medical Research Acquisition Activity Cooperative Agreement, as well as made possible by a research grant that was awarded and administered by the U.S. Army Medical Research & Materiel Command (USAMRMC) and the Telemedicine & Advanced Technology Research Center (TATRC), at Fort Detrick, MD. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p><p><a name="cor1"></a>* E-mail: <a href="mailto:udemirci@rics.bwh.harvard.edu">udemirci@rics.bwh.harvard.edu</a> (UD); <a href="mailto:lhsiao@partners.org">lhsiao@partners.org</a> (L-LH)</p></div>





<div id="section1" xpathlocation="/article[1]/body[1]/sec[1]"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3 xpathlocation="noSelect">Introduction&nbsp;<a href="#top">Top</a></h3><p xpathlocation="/article[1]/body[1]/sec[1]/p[1]">For stem cell characterization, understanding single cell level functional genomics has become increasingly important <a href="#pone.0017455-Marson1">[1]</a>, <a href="#pone.0017455-Bianco1">[2]</a>, <a href="#pone.0017455-Takahashi1">[3]</a>, <a href="#pone.0017455-Zipori1">[4]</a>, <a href="#pone.0017455-Slack1">[5]</a>, <a href="#pone.0017455-Fan1">[6]</a>, <a href="#pone.0017455-Kurimoto1">[7]</a>, <a href="#pone.0017455-Svahn1">[8]</a>. As new regenerative therapies using tissue engineering <a href="#pone.0017455-Bianco1">[2]</a> emerge, the existence of tissue-specific stem cells in adult organs has extensively been investigated in bone marrow, skin, heart, muscle, pancreas, lungs, and the nervous system. However, the characterization of differentiated progeny has been hampered by the lack of cell markers and low viability of the purified cells <a href="#pone.0017455-Fabbri1">[9]</a>. For instance, single cell transplantation methods can significantly benefit from efficient cell isolation and handling techniques <a href="#pone.0017455-Leong1">[10]</a>. Recent advances in mRNA amplification and cell sorting technologies offer insights into single cell genomics <a href="#pone.0017455-Marson1">[1]</a>, <a href="#pone.0017455-Bianco1">[2]</a>, <a href="#pone.0017455-Takahashi1">[3]</a>, <a href="#pone.0017455-Zipori1">[4]</a>, <a href="#pone.0017455-Slack1">[5]</a>, <a href="#pone.0017455-Fan1">[6]</a>, <a href="#pone.0017455-Walker1">[11]</a>. However, single cell level genomic studies require amplification by a factor of a billion to reach detection levels from a few femtograms of mRNA present in a single cell. Therefore, to accurately profile single cell genes from a heterogeneous cell solutions or a tissue sample, it is essential to minimize RNA contamination from surrounding cells by enriching the fraction of the target cell type.</p>
<p xpathlocation="/article[1]/body[1]/sec[1]/p[2]">Cell pattering and encapsulation in droplets is a challenging and exciting field with multiple possible applications including tissue printing <a href="#pone.0017455-Sun1">[12]</a>, <a href="#pone.0017455-Mironov1">[13]</a>, cell sorting <a href="#pone.0017455-Baret1">[14]</a>, and cryobiology <a href="#pone.0017455-Song1">[15]</a>. Several approaches have been developed to isolate single cells (<a href="#pone-0017455-t001"><strong>Table 1</strong></a>). The most common methods are microscale cell manipulation <a href="#pone.0017455-Kvist1">[16]</a>, serial dilutions of a culture or a co-culture of cells <a href="#pone.0017455-Zhang1">[17]</a>, laser capture microdissection (LCM) <a href="#pone.0017455-Geigl1">[18]</a>, and fluorescence-activated cell sorting (FACS) <a href="#pone.0017455-Raghunathan1">[19]</a>. These methods have challenges with complexity, time consumption and inefficiencies in isolating cells that are contamination-free and viable. In addition, histological methods can damage mRNA both in frozen and paraffin-embedded sections <a href="#pone.0017455-Farragher1">[20]</a>. Traditional FACS and LCM require large sample volumes (milliliters), and utilize expensive instruments used by skilled operators (<a href="#pone-0017455-g001"><strong>Fig. 1a</strong></a>). FACS can sort cells, at a single cell level in nanoliter volumes. However, FACS does not pattern these cells. Recently, these traditional technologies have been modified and adapted towards microfluidics <a href="#pone.0017455-Zeng1">[21]</a>, <a href="#pone.0017455-DiCarlo1">[22]</a>, <a href="#pone.0017455-Ino1">[23]</a>, <a href="#pone.0017455-White1">[24]</a>, <a href="#pone.0017455-Ottesen1">[25]</a>, <a href="#pone.0017455-Toriello1">[26]</a>, <a href="#pone.0017455-Love1">[27]</a>. Microengraving <a href="#pone.0017455-Love1">[27]</a>, <a href="#pone.0017455-Lin1">[28]</a> has low complexity; it rapidly loads individual cells, and creates low mechanical stress during cell loading. However, it suffers from limited control over number of cells per well due to manual cell loading process. These new technologies have potential for single cell genomic analysis of target cells, <em>e.g.</em> stem cells and uncultured organisms <a href="#pone.0017455-Walker1">[11]</a>. However, these capabilities arrive at a substantial cost in increased design complexity, development cost <a href="#pone.0017455-Fu1">[29]</a>. Further, there are challenges to control the number of cells deposited to a predetermined location. The microfluidic systems are also great tools to handle and sort cells. Although the cell handling processes have been simplified in microfluidic systems, cell tracking for sorting on chip still requires peripheral setups followed by subsequent cell separation steps. As the heterogeneity of sample increases, the types of cells that need to be tracked in the sample also increase. The outcome is that the tracking system requires more complex peripheral setups and it can require high-end computerized controls which has an impact on scalability. A great example of such systems is best demonstrated by Quake and Hong <a href="#pone.0017455-Fu1">[29]</a>, <a href="#pone.0017455-Hong1">[30]</a>, <a href="#pone.0017455-Pushkarev1">[31]</a>, <a href="#pone.0017455-Marcus1">[32]</a>. To address these challenges, we developed a simple, high-throughput platform for single cell isolation with direct access to patterned cells (<a href="#pone-0017455-g001"><strong>Fig. 1b</strong></a>). The methodology is based on a &ldquo;drop-on-demand&rdquo; cell patterning technology that follows simple random sampling (SRS) <a href="#pone.0017455-Ross1">[33]</a>, <a href="#pone.0017455-Sa1">[34]</a>, <a href="#pone.0017455-Grinstead1">[35]</a>. The system patterned an array of 10&times;10 droplets that encapsulated single cells from a heterogeneous cell mixture at a high-throughput of within 4 seconds. Subsequently, imaging systems with a wide field of view are available to monitor the patterned droplet array within a few seconds showing positions of target cells <a href="#pone.0017455-Moon1">[36]</a>. Since the droplets are printed onto a glass surface, each encapsulated cell of interest can then be accessed freely.</p>
<div class="figure" xpathlocation="/article[1]/body[1]/sec[1]/fig[1]"><a name="pone-0017455-g001" id="pone-0017455-g001" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.g001" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[1]/fig[1]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.g001" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Figure 1. </span></a> <span xpathlocation="/article[1]/body[1]/sec[1]/fig[1]/caption[1]/title[1]">Description of a conventional procedure for total RNA expression analysis versus the technique described.</span></strong></p><p xpathlocation="/article[1]/body[1]/sec[1]/fig[1]/caption[1]/p[1]">(<strong>a</strong>) Conventional single cell isolation method for total RNA expression analysis. 1) Heterogeneous sample was collected from a specific tissue, macro-dissection, 2) cells were stained with a specific antibody, 3) target cells were collected with conventional FACS, 4) multiple dilution steps generated a small population of target cells, 5) total RNA gene expression analysis with microarray. (<strong>b</strong>) Drop-on-demand total RNA analysis approach. 1) Heterogeneous sample was collected, 2) cells were stained with antibody and patterned with cell-encapsulating droplets, 3) specific homogeneous samples containing target cells were produced by a cell droplet patterning platform; homogeneous samples were identified by an automated imaging system, 4) total RNA gene expression analysis with microarray.</p>
<span xpathlocation="noSelect">doi:10.1371/journal.pone.0017455.g001</span><div class="clearer"></div></div><div class="figure" xpathlocation="/article[1]/body[1]/sec[1]/table-wrap[1]"><a name="pone-0017455-t001" id="pone-0017455-t001" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.t001" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject_002.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[1]/table-wrap[1]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.t001" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Table 1. </span></a> <span xpathlocation="/article[1]/body[1]/sec[1]/table-wrap[1]/caption[1]/title[1]">Performance comparison with conventional methods: isolating single cell's from a heterogeneous cell mixture.</span></strong></p><span xpathlocation="noSelect">doi:10.1371/journal.pone.0017455.t001</span><div class="clearer"></div></div><p xpathlocation="/article[1]/body[1]/sec[1]/p[3]">In this paper, we experimentally and theoretically analyzed the cell encapsulation process in microdroplets. We analyzed the cell encapsulating droplets for cell viability and performed genomic analysis on the printed stem cells and compared them to the non-patterned controls. Here, we present the first time genomic analysis performed on cells patterned using cell printing indicating that the cells are genomically functional through the printing process compared to the controls. We used immunostaining prior to patterning to distinguish target cells within the heterogeneous population. Based on the optical images, each patterned droplet fell into one of the four categories: a droplet containing (i) a single target cell, (ii) no cells (empty droplet), (iii) a non-target cell, or (iv) multiple cells (of either the target and non-target cell type) (<a href="#pone-0017455-g002"><strong>Fig. 2b</strong></a>). Then, we performed RNA extraction from cells within nanoliter droplets for genomic analysis. This permits high-throughput processing and reduction in time required for isolation of cells. Such novel methods can enable rapid biomarker studies in systems biology research <a href="#pone.0017455-Megason1">[37]</a>.</p>
<div class="figure" xpathlocation="/article[1]/body[1]/sec[1]/fig[2]"><a name="pone-0017455-g002" id="pone-0017455-g002" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.g002" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject_003.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[1]/fig[2]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.g002" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Figure 2. </span></a> <span xpathlocation="/article[1]/body[1]/sec[1]/fig[2]/caption[1]/title[1]">Patterned droplets.</span></strong></p><p xpathlocation="/article[1]/body[1]/sec[1]/fig[2]/caption[1]/p[1]">(<strong>a</strong>) Schematic droplet subarray showing four different cases of patterned droplets. Green colored circles indicated target cells. (<strong>b</strong>) Portion of a droplet array showing examples of each of the four different cases of patterned droplets, 1) single target cell droplet, 2) an empty droplet without cells, 3) a non-target cell droplet, and 4) a heterogeneous droplet containing target and non-target cells. Green fluorescent stained cells were target cells. Scale bars, 200 &micro;m.</p>
<span xpathlocation="noSelect">doi:10.1371/journal.pone.0017455.g002</span><div class="clearer"></div></div></div>

<div id="section2" xpathlocation="/article[1]/body[1]/sec[2]"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3 xpathlocation="noSelect">Results and Discussion&nbsp;<a href="#top">Top</a></h3><p xpathlocation="/article[1]/body[1]/sec[2]/p[1]">We evaluated our drop-on-demand single cell patterning platform by measuring droplet volume, number of cells encapsulated per droplet for different cell loading concentrations and target cell content. Droplet volume was measured by two independent techniques: stroboscopic imaging in air and wetting contact angle measurements on a glass surface. Stable droplet generation at 25 Hz was achieved by employing a 34.4 kPa ejection pressure at 1.25 mPa&middot;s viscosity. Under these conditions, the droplet diameter in air depended on the valve opening duration. The average droplet diameters in air were 220&plusmn;9 &micro;m, 245&plusmn;12 &micro;m, and 285&plusmn;18 &micro;m, corresponding to valve opening durations of 50 &micro;s, 55 &micro;s, and 60 &micro;s, respectively. A 55 &micro;s valve opening duration was chosen as the minimum value required to avoid clogging and formation of satellite droplets. We printed 50&times;50 array of droplets at a rate of 25 droplets per second. The array density was one droplet per mm<sup>2</sup> with a 525&plusmn;15 &micro;m droplet spread diameter on the surface. This droplet spread diameter depended on the surface tension (28.3&deg; contact angle on a glass slide) and droplet volume (7.6&plusmn;0.6 nanoliter per droplet; see <a href="#s3"><strong>Materials and Methods</strong></a>, <em>Drop-on-demand single cell patterning</em>).</p>
<p xpathlocation="/article[1]/body[1]/sec[2]/p[2]">Using these droplet generation parameters, we investigated the effect of the cell loading concentration on the patterned droplets. We obtained both experimental values and developed statistical models that describe the cell encapsulation process. Heterogeneous mixtures of mouse embryonic stem cells (mESCs) were prepared consisting of 10% to 50% stained target stem cells. The average number of target cells per droplet was calculated over 100 droplets (see <a href="#s3"><strong>Materials and Methods</strong></a>, <em>Cell preparation and staining</em>). We tested cell loading concentrations ranging from a 0.5&times;10<sup>5</sup> to 7.5&times;10<sup>5</sup> cells/ml. Our theoretical and experimental results indicated that the 1.0&times;10<sup>5</sup> cells/ml concentration was optimal to encapsulate single cells in droplets, since the resulting average number of cells per droplet and the standard deviation were 0.88&plusmn;0.14 (<a href="#pone-0017455-g003"><strong>Fig. 3a</strong></a>). The model (see <strong><a href="#pone.0017455.s005">Fig. S5a</a></strong> online) agrees with the experimental results <a href="#pone-0017455-g003"><strong>Fig. 3a</strong></a>. These both indicate that the maximum single cell encapsulation event is likely to take place at the 1&times;10<sup>5</sup> concentration. These statistics correspond to the case of single cell encapsulation within a droplet (with 36.8% probability), followed by other cases of an empty droplet (<a href="#pone-0017455-g003"><strong>Fig. 3e</strong></a>, 36.8%), and a droplet containing two cells (<a href="#pone-0017455-g003"><strong>Fig. 3e</strong></a>, 18.4%). Further, the number of cells per droplet was found to be independent of the target cell fraction (<a href="#pone-0017455-g003"><strong>Fig. 3a</strong></a>, 10&ndash;50%) in the heterogeneous cell mixture. We observed that the standard deviation for the number of encapsulated cells per droplet was higher than &plusmn;1.5 cells when the cell loading concentrations exceeded 2.5&times;10<sup>5</sup> cells/ml (<a href="#pone-0017455-g003"><strong>Fig. 3b</strong></a>). However, the number of cells per droplet obtained by merging multiple droplets by ejection to the same location showed smaller standard deviation compared to ejecting a single droplet at a higher cell loading concentration (less than &plusmn;0.7 cells/droplet, <a href="#pone-0017455-g003"><strong>Fig. 3b</strong></a>). This result is due to a small cell-to-droplet volume fraction of 1.7% using 7.6 nanoliter droplets. As we increase number of droplets, the sampling process fulfills the law of large numbers (LLN, n = 106, 90% confidence level and 15% tolerance), the cell encapsulation mechanism can be treated as a random cell sampling process from a large population of cells, <em>i.e.</em> Student's t-test and single factor analysis of variance (ANOVA). These statistical results as shown in <a href="#pone-0017455-g003"><strong>Fig. 3</strong></a> indicate that our random cell encapsulation and patterning methods follow the central limit theorem (CLT) <a href="#pone.0017455-Ross1">[33]</a>, <a href="#pone.0017455-Sa1">[34]</a>, <a href="#pone.0017455-Grinstead1">[35]</a>. This allows us to conclude that the average and standard deviation of sampled droplets can be used for statistical estimation of entire sample volume. This approach may provide a powerful tool for sampling single target cells in a high-throughput manner without searching the entire cell population in a reservoir.</p>
<div class="figure" xpathlocation="/article[1]/body[1]/sec[2]/fig[1]"><a name="pone-0017455-g003" id="pone-0017455-g003" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.g003" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject_004.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[2]/fig[1]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.g003" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Figure 3. </span></a> <span xpathlocation="/article[1]/body[1]/sec[2]/fig[1]/caption[1]/title[1]">Homogeneous droplet analyses.</span></strong></p><p xpathlocation="/article[1]/body[1]/sec[2]/fig[1]/caption[1]/p[1]">(<strong>a</strong>) Droplets are generated at cell concentrations: 0.5&times;10<sup>5</sup>, 1.0&times;10<sup>5</sup>, 1.5&times;10<sup>5</sup>, and 2.0&times;10<sup>5</sup> cells/ml and they contained 0.28&plusmn;0.12, 0.88&plusmn;0.14, 1.50&plusmn;0.10, and 2.20&plusmn;0.19 cells per droplet, respectively (mean &plusmn; standard deviation). The droplets were ejected using two controlled parameters: 34 kPa nitrogen gas pressure and 55 &micro;s valve opening period. Values were obtained from 5 groups, each containing 100 droplets. Each group was tested by single factor analysis of variance (ANOVA). F(3,16) = 0.15, 0.58, 1.22, and 0.16, n = 5 mean values per each group, *p&gt;0.33. (Student's t-test between target cell concentrations: **p&lt;0.0004, n = 5 mean values of 4 concentrations). (<strong>b</strong>) Number of cells in droplets was obtained at cell concentrations: 2.5&times;10<sup>5</sup>, 5.0&times;10<sup>5</sup>, and 7.5&times;10<sup>5</sup> cells/ml. Regression coefficients, R<sup>2</sup> values, were *0.93 (1.7 cells/drop), **0.96 (3.6 cells/drop), and ***0.92 (6.6 cells/drop), n = 28 droplets. (<strong>c</strong>) Comparing target cell concentration in the reservoir and in the droplet pattern. Average and standard deviation of 10% to 50% target cell concentration in the reservoir were 10.3&plusmn;0.6, 19.0&plusmn;1.0, 27.8&plusmn;2.0, and 45.1&plusmn;2.1. F(3,16) = 0.82, 1.38, 0.98, and 0.52, *p&gt;0.29, ANOVA for each cell concentration (student's t-test for each target cell concentration: **p&lt;0.001, n = 5 mean values of 4 different target cell concentrations). (<strong>d</strong>) Fraction of empty droplets in droplet array as a function of cell concentration. Average and standard deviation of empty droplets for 0.5 to 2.5&times;10<sup>5</sup> cells/ml concentrations were 72.8&plusmn;12.8%, 40.6&plusmn;7.7%, 23.8&plusmn;4.0%, 12.7&plusmn;5.8%, and 2.2&plusmn;0.3%, respectively. F(3,16) = 0.05, 0.64, 0.56, 0.05, and 0.67, *p&gt;0.58, ANOVA for each target cell concentration (student's t-test for each cell concentration: **p&lt;0.016, n = 5 mean values of 4 different target cell concentrations). (<strong>e</strong>) Fraction of target homogeneous droplets among the patterned droplets was calculated as the ratio between the number of cells in homogenous droplets and the overall number of droplets for different cases: 1, 2, 3, 4, and 5 cells per droplet. This excluded the empty droplets. At 1.0&times;10<sup>5</sup>, for the case of one cell per droplet, the average and standard deviation values of homogeneity were 9.5&plusmn;1.4%, 18.8&plusmn;3.7%, 26.0&plusmn;3.1%, and 45.5&plusmn;8.8% for 10 to 50% target cell concentration. The fractions were 0.4&plusmn;0.9%, 3.1&plusmn;2.1%, 5.1&plusmn;4.2%, and 9.6&plusmn;4.8% for droplets containing two cells. (<strong>f</strong>) Frequency of homogeneous droplets based on cell concentration. This analysis includes the empty droplets. Among the different target cell concentrations, the 1&times;10<sup>5</sup> cells/ml concentration showed 4.7&plusmn;0.6%, 9.0&plusmn;1.9%, 12.7&plusmn;1.6%, and 21.6&plusmn;5.2% homogeneous droplet occurrences. F(1,8) = 22.7, 11.2, and 13.3, *p&lt;0.01, ANOVA for 1&times;10<sup>5</sup> cells/ml through 10 to 50% target cell concentration (student t-test: **p = 0.12, ***p&lt;0.05, n = 5). Error bars represented the standard deviation of the mean. The p values in student's t-test were calculated based on two-sided distributions with unequal variances.</p>
<span xpathlocation="noSelect">doi:10.1371/journal.pone.0017455.g003</span><div class="clearer"></div></div><p xpathlocation="/article[1]/body[1]/sec[2]/p[3]">We investigated the viability of the cells patterned in droplets. The reservoir cell viability was 96.7&plusmn;0.5% using a 10 &micro;l volume at 7.5&times;10<sup>5</sup> cells/ml. Following the 1.7 minutes of patterning process using a 19 &micro;l sample, cell viability was observed to be 93.8&plusmn;1.1% in a 50&times;50 array of cell encapsulating droplets. When the optimal cell concentration for single cell patterning (1.0&times;10<sup>5</sup> cells/ml) was used, the patterned cell viability was 97.0&plusmn;0.8%. The relatively higher patterning viability is due to reduced droplet packing density, which minimizes possible exposure to mechanical shear forces at the valve during cell encapsulation. Underlying this high viability was the low (1.7%) cell-to-droplet volume fraction we used during encapsulation process. Furthermore, we confirmed that patterned cell viability does not depend on the target cell concentration in the reservoir (see <strong><a href="#pone.0017455.s004">Fig. S4</a></strong> online).</p>
<p xpathlocation="/article[1]/body[1]/sec[2]/p[4]">Based on our statistical analysis (see <a href="#s3"><strong>Materials and Methods</strong></a> <em>Statistical Modeling</em>), the single target cell encapsulation process followed Poisson distribution and matched with 90% confidence level and 15% tolerance. These results show that our system can be used to encapsulate single target cells from a heterogeneous solution that has 9&plusmn;1% target cells by using only 10&times;10 array of droplets. This reduces time to reach to target cells without searching the entire sample volume in the ejection reservoir.</p>
<p xpathlocation="/article[1]/body[1]/sec[2]/p[5]">We validated that our patterning method produces droplet arrays that conform to CLT by comparing the fraction of target cells in the reservoir to the fraction in the patterned droplet array. The comparison is conducted using 10&times;10 droplet array subset randomly chosen from a 50&times;50 patterned droplet array. The fraction of target cells in the reservoir cell mixture (10%) was essentially mimicked (10&plusmn;2.2%) in the patterned droplet subarrays (<a href="#pone-0017455-g003"><strong>Fig. 3c</strong></a>). The other heterogeneous samples (20%, 30%, and 50%) agreed with this result. Therefore, evaluating a 10&times;10 droplet array allowed us to infer the target cell concentration in the reservoir. The fraction of empty droplets was also investigated with concentrations ranging from 1.0&times;10<sup>5</sup> to 2.5&times;10<sup>5</sup> cells/ml. Percentage of empty droplets determines the expected yield when searching for target cells (<em>i.e.</em>, cells to be isolated for further mRNA extraction and analysis, <a href="#pone-0017455-g003"><strong>Fig. 3d</strong></a>). At a low cell concentration, 0.5&times;10<sup>5</sup> cells/ml, the probability of patterning empty droplets was highest (72.8&plusmn;12.8%). We choose 1.0&times;10<sup>5</sup> cells/ml concentration to pattern single cell encapsulating droplets. The fraction of empty droplets at this concentration ranged from 32.9% to 48.3% over the range of cell mixture compositions studied. Despite these values, high-throughput target cell isolation was achieved with the automated rapid patterning capability.</p>
<p xpathlocation="/article[1]/body[1]/sec[2]/p[6]">The patterning process samples 10&times;10 of droplets (of total volume 0.76 &micro;L) out of the total reservoir volume of 100 &micro;L. While characterizing the statistics governing the patterned target cell droplets printed from a heterogeneous cell mixture, we observed agreement (0.9%~7.0% difference) between the reservoir mixture composition and the composition observed in patterned subarrays for 10% to 50% target cell containing mixtures (<a href="#pone-0017455-g003"><strong>Fig. 3e</strong></a>). Starting from a cell reservoir containing a 10% fraction of target cells, a target homogeneous droplet fraction of 9.5&plusmn;1.4% was observed in a 10&times;10 droplet subarray at a reservoir cell density of 1.0&times;10<sup>5</sup> cells/ml. Experimental results indicate that isolation of &ldquo;a droplet containing a single target cell&rdquo; within a 10&times;10 array is likely (at &gt;90% confidence level) for target cell fractions down to 1% (<a href="#pone-0017455-g003"><strong>Fig. 3e</strong></a>). This lower limit of sampling fraction, 1% v/v, assumes the empty droplet percentage to be at the maximum value that we experimentally observed to be as 48.3% at 1.0&times;10<sup>5</sup> cells/ml concentration. Moreover, when a droplet contained more than three cells, the fraction of droplets containing only target cells became zero in the 10&times;10 droplet subarray. We also experimentally observed the highest probability to pattern homogeneous droplets at the reservoir concentration of 1.0&times;10<sup>5</sup> cells/ml. At the 1.0&times;10<sup>5</sup> cells/ml reservoir concentration, the values for homogeneous droplet occurrences were 4.7&plusmn;0.6%, 9.0&plusmn;1.9%, 12.7&plusmn;1.6%, and 21.6&plusmn;5.2% for 10%, 20%, 30%, and 50% target cell concentrations, respectively (<a href="#pone-0017455-g003"><strong>Fig. 3f</strong></a>). Our results demonstrate that higher single cell encapsulation probability yields higher homogeneity as dictated by Poisson distribution and shown by experimental results.</p>
<p xpathlocation="/article[1]/body[1]/sec[2]/p[7]">Finally, we evaluated the sensitivity and reproducibility of a functional genomic analysis of stem cell encapsulating droplets. For these evaluations, we used DNA microarrays to measure gene expression levels in RNA samples extracted from mouse embryonic stem cells obtained by using drop-on-demand. We compared the results with RNA extracted from a control pool of mESCs, which were isolated with serial dilution and manual pipetting. The total RNA quality from the droplet-based isolated cells was similar to that of the control when assessed with the Agilent bioanalyser using the 28s/18s ratio (<a href="#pone-0017455-g004"><strong>Fig. 4a</strong></a>). This demonstrated that RNA remains intact throughout the patterning process. In a genome-wide analysis, the 1000 genes with the highest expression levels were measured on DNA microarrays for the printed and control cells. The reproducibility of the gene expression is illustrated by the scatter plots of the microarray measurements on replicate samples (<a href="#pone-0017455-g004"><strong>Fig. 4b, c</strong></a>). Although the expression levels exhibited greater variability at the lower expression levels than the control samples, the median coefficient of variation (CV) across three replicates was 5% in both cases. Stem cell-related markers were utilized to assess whether the RNA obtained from the printed cell group provided useful biological information in comparison to the control samples. Altogether eleven stem cell markers including Kit and Notch1 were found in both the printed cells and non-printed control groups (<a href="#pone-0017455-g004"><strong>Fig. 4d</strong></a>). These results indicate that we were able to successfully isolate and analyze mRNA from cell encapsulating droplets for functional genomic studies. (See <a href="#s3"><strong>Materials and Methods</strong></a>, <em>Total RNA extraction and analysis</em>).</p>
<div class="figure" xpathlocation="/article[1]/body[1]/sec[2]/fig[2]"><a name="pone-0017455-g004" id="pone-0017455-g004" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.g004" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject_005.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[2]/fig[2]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0017455&imageURI=info:doi/10.1371/journal.pone.0017455.g004" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Figure 4. </span></a> <span xpathlocation="/article[1]/body[1]/sec[2]/fig[2]/caption[1]/title[1]">Total RNA expressions in printed droplets (virtual gel).</span></strong></p><p xpathlocation="/article[1]/body[1]/sec[2]/fig[2]/caption[1]/p[1]">(<strong>a</strong>) Total RNA quality on virtual gels (Agilent bioanalyser) as number of droplets increase. The 96-well plate was then placed underneath the ejector and one droplet was ejected into each of the first nine wells of the first row of wells. This procedure was repeated with additional four rows of the plate, where two, three, four and five droplets were ejected in each well resulting to a total of five groups of droplet samples, each group having an increasing droplet volume and increasing number of cells ranging from ten to fifty cells. For extraction purposes and consistency, before the RNA analysis was performed, each row was divided into three triplicates. Hence, the analysis was based on average 30, 60, 90, 120, and 150 cells for each triplicate for bioanalysis. To investigate nonprinted cells, 0.5 &micro;l samples were drawn from the cell solution and pipetted into the three wells in each of the first three rows of the plate as controls. The controls had 1000 cells on average. The last lane showed the results for the control RNA which was prepared by a manual dilution method without ejection. (<strong>b</strong>, <strong>c</strong>) Comparison of the reproducibility of the RNA expression levels for (<strong>b</strong>) two replicates of the control set by the manual pipette method and (<strong>c</strong>) two experimental RNA samples obtained by droplet method. The scatter-plots were compared the reproducibility of expression level measurements for the 1000 genes with the highest expression levels in the experimental samples. (<strong>d</strong>) Expression of stem cell-related markers was examined to assess whether RNA obtained from the cell encapsulating droplets provided useful biological information in comparison to control samples. Our results showed that these 11 stem cell markers including Kit and Notch1, were found in both the patterned cells and the control groups.</p>
<span xpathlocation="noSelect">doi:10.1371/journal.pone.0017455.g004</span><div class="clearer"></div></div><p xpathlocation="/article[1]/body[1]/sec[2]/p[8]">There are also limitations on the drop-on-demand approach for single cell sorting compared to the microfluidic deterministic cell encapsulation approaches. As the heterogeneity of sample increases, the types of cells that need to be tracked in the sample also increase for microfluidic systems. The outcome is that the tracking system requires more complex peripheral setups and it can require high-end computerized controls which has an impact on scalability <a href="#pone.0017455-Fu1">[29]</a>, <a href="#pone.0017455-Hong1">[30]</a>, <a href="#pone.0017455-Pushkarev1">[31]</a>. The drop-on-demand cell encapsulation approach presented here has a trade-off from a deterministic aspect, but on the other hand it offers lesser handling steps to sort and pattern cells, where cells might be affected by the conditions in the physical environment. For single cell encapsulation, microfluidic method is more deterministic and may provide a better control over cell encapsulation. Further, the deterministic approach offers an efficient system to handle minimum number of cells, i.e. single to ten cells. Second, microfluidic approach can be convenient in case an integrated on-chip experiment requires further cell handling steps after sorting such as on-chip polymerase chain reactions (PCR). Finally, throughput of drop-on-demand sorting process is limited by the parallel printing setup and the wide field of view imaging method.</p>
<p xpathlocation="/article[1]/body[1]/sec[2]/p[9]">We demonstrated a high-throughput drop-on-demand single cell isolation technology and presented how it fits to the statistical models. Our data revealed that the drop-on-demand single cell patterning platform can isolate viable target stem cells from a heterogeneous sample. In addition, this patterning approach can be adapted to generate multiple droplet arrays in parallel, further enhancing throughput. This work marks the first genomic analysis study on printed cells. Furthermore, we showed that functional genomic information, specifically mRNA expression levels, obtained from ejected cells was preserved throughout the entire process.</p>
</div>

<div id="section3" xpathlocation="/article[1]/body[1]/sec[3]"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3 xpathlocation="noSelect">Materials and Methods&nbsp;<a href="#top">Top</a></h3>
<h4 xpathlocation="/article[1]/body[1]/sec[3]/sec[1]/title[1]">Cell preparation and staining</h4>
<p xpathlocation="/article[1]/body[1]/sec[3]/sec[1]/p[1]">The work space is a HEPA filter equipped sterile hood (Cleanroom International, 13202). All materials were decontaminated with 70% ethanol and RNaseZAP&reg; (Applied Biosystems) prior to introducing the cell solutions. Mouse embryonic stem cells (mESCs, wild type R1 cell line, ~20% confluent) <a href="#pone.0017455-Hwang1">[38]</a> were trypsinized (10&times;0.5 Trypsin-EDTA, Gibco, 15400) and passaged from Corning&reg; flask (Corning, CLS3150) into a Falcon&trade; tube (BD, 352096). The cell solution was centrifuged at 1000 rpm for 3 minutes and cells were washed with DPBS (Dulbecco's phosphate buffered saline, Dulbecco, P1010) and resuspended in culture medium. The culture medium was prepared from 500 ml Knockout ES Media (Gibco, 11965-092), 50 ml ES FBS (Gibco, 10439-024), 1 ml 2-Mercaptoethanol (Sigma, P4333), 5 ml NEAA (Non-essential amino acid, Sigma, M7145), 5 ml L-Glutamin (Gibco, 25030), 2 &micro;l LIF/ml (Leukemia inhibitory factor, Millipore, ESGRO&reg;), and 5 ml Pen/Strep (Sigma, P4333). All components were passed through a sterile filter (500 ml Express Plus 0.22 &micro;m membrane, Millipore, SCGPU05RE). A sample/aliquot of the cell solution was stained with 0.4% Trypan Blue solution (Invitrogen, 15250061) and counted with a hemacytometer (Hausser Scientific, 1483). Two sets of low (1&times;10<sup>5</sup>, 1.5&times;10<sup>5</sup>, and 2&times;10<sup>5</sup> cells/ml) and high (2.5&times;10<sup>5</sup>, 5.0&times;10<sup>5</sup>, and 7.5&times;10<sup>5</sup> cells/ml) cell concentrations were prepared. Using a molecular probes Live/Dead assay (Invitrogen, L-2334) for mammalian cells, pre- and post-ejection cell viabilities were recorded. Also, to make a heterogeneous cell mixture that contains both stained and non-stained cells, two samples of each concentration of cells were prepared and stored in separate test tubes. After staining with a Live/Dead assay and washing with DPBS to remove excessive staining solution, the stained and unstained cells were mixed to prepare 10% to 50% volume fraction heterogeneous solutions. These solutions served as models that represent different fractions of target cells. Pre-ejection cell viability was measured from samples taken directly from the cell solutions (see <strong><a href="#pone.0017455.s004">Fig. S4</a></strong> online). Post-ejection cell viability was measured from droplets ejected through the 150 &micro;m valve orifice with a frequency ranging from 1 Hz to 1 kHz. The pulse duration was 55 &micro;s to 65 &micro;s, at 34.5 kPa of gas pressure.</p>


<h4 xpathlocation="/article[1]/body[1]/sec[3]/sec[2]/title[1]">Drop-on-demand single cell patterning</h4>
<p xpathlocation="/article[1]/body[1]/sec[3]/sec[2]/p[1]">Drop-on-demand can generate droplets both with spatial control over position and temporal control over ejection. The single cell droplet ejector system consists of an automated xyz stage (NLS4 Series Precision Linear Stage, Newmark systems Inc.) and a sub-nanoliter dispensing valve (TechElan LLC, G100), which were synchronized with a control program, Labview&trade; (Labview, National Instruments Corporation) (see <strong><a href="#pone.0017455.s001">Fig. S1a</a></strong> online). The spatial resolution and repeatability of the stage are 0.13 &micro;m and 5 &micro;m, respectively. To image <em>in-situ</em> cell encapsulating droplets during the ejection process, a CCD camera (Edmund optics, EO-1312M) equipped with a 0.5&times; lens (INFINITUBE FM-200 (NT58-309, 2&times;) and a PL/FD-195 OBJ objective (NT59-115, 0.25&times;, Edmund optics) was combined with a synchronized light controller (S4000 strobe controller, Edmund optics) for stroboscopic illumination. The overall system, i.e. xyz stage, sub-nanoliter dispensing valve, and stroboscopic light controller, was synchronized and programmed by a PC-based DAQ control board (NI cDAQ-9172 and NI-9401, National Instruments). Pressure regulated nitrogen gas was connected to a syringe reservoir through an adapter cap (KDS503S6, Techni-Tool Inc.), and the syringe was connected to the sub nanoliter dispensing valve by tubing (Tygon&reg; tube, Fisher scientific Inc.).</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/sec[2]/p[2]">A heterogeneous sample comprising a mixture of cells and cell media solution was loaded into a 1 ml syringe reservoir and each droplet was printed at pre-determined positions on a glass surface or into a 96-well plate. The number of cells per droplet was primarily determined by the droplet size and the cell concentration in the printing solution. Prior to taking a 100 &micro;L cell solution, the Falcon tube was vortexed. The reservoir was stirred also after loading samples to maintain an even cell distribution within the medium. High-frequency actuation (1 kHz) was used to eject the excess cell solution, after which the valve was washed out with DI water (~60&deg;C). We calibrated droplet size and maintained ejection stability to ensure that the orifice was free of cell debris and medium residues. Once this process was completed, 70% ethanol was ejected through the valve, leaving a small amount of ethanol in the syringe and valve until the next experiment. This ensured clean, repeatable, and reliable operation. The size of each droplet was determined by stroboscopic images during stable droplet formation. Adjustable ejection parameters were xyz stage speed, valve opening frequency and duration, and gas pressure. The valve opening time and gas pressure were regulated to control the droplet size and number of cells per droplet. A droplet size measurement was conducted both in air and on the glass substrate to provide feedback to the manual/automatic droplet size controller. Droplet stability was tested using 90 &micro;s valve opening time and 34.5 kPa pressurized nitrogen gas. Stroboscopic images of ejected droplets were taken every second (see <strong><a href="#pone.0017455.s002">Fig. S2a</a></strong> online). If the ejector was not operated in a stable regime, satellite droplets were generated during ejection. Control over stable droplets without satellites was maintained at optimized ejection conditions, <em>i.e.</em> 64 &micro;s valve opening time, 34.4 kPa pressurized nitrogen gas, and 25 Hz ejection frequency (see <strong><a href="#pone.0017455.s002">Fig. S2b</a></strong> online). The size of the ejected droplets was investigated in air and at the receiving surface of a slide glass substrate by ejecting 100 droplets. Three pressure and ejection conditions were used at 34.4 kPa of air pressure for pulse durations of 55 &micro;s, 60 &micro;s and 65 &micro;s, yielding droplets of different sizes. As shown in <a href="#pone-0017455-g002"><strong>Figure 2b</strong></a>, we ejected multiple droplets under same ejection conditions to investigate the droplet size uniformity. A liquid nitrogen bath was prepared by inserting a 60 mm (D)&times;15 mm (H) Petri dish inside a Styrofoam box, filling the dish and surrounding box area with liquid nitrogen, and placing the box 20 mm underneath the valve. A fluorescent microscope (Eclipse TE-2000 U, Nikon) equipped with a CCD camera (Spot RT-KE Mono, RT700, Diagnostic Instruments, Inc.) was paired with a software (Spot Advanced, Diagnostic Instruments, Inc.) to capture images of the frozen droplets at 10&times; and 20&times; magnification. Droplet diameters were obtained with the software by fitting circles around each droplet image (Spot Advanced). To measure the diameter of the ejected droplets on the surface (see <strong><a href="#pone.0017455.s002">Fig. S2c</a></strong> online), the same stroboscopic image setup was used after changing the lens to a 0.25&times; magnification tube (INFINITUBE FM-200 (NT58-309, 2&times;) and a PL/FD-390 OBJ objective (NT59-117, 0.125&times;), Edmund optics).</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/sec[2]/p[3]">Droplet arrays (50&times;50 in size) were patterned at 25 Hz at 34.5 kPa of nitrogen gas pressure onto a 75 mm&times;50 mm glass microscope slide. The distance between the valve ejector and the slide was 1.5 mm. The drop-to-drop distance was determined to be larger than the droplet size to avoid overlapping patterns. The microscope slide was then placed inside a covered 100 mm (diameter)&times;15 mm (height) Petri dish. To avoid droplet evaporation while imaging, PBS surrounding the patterned droplet area was used to create a local humid environment. The droplet array was searched for target cells using a fluorescent microscope with an automatic stage (Axio observer equipped with Axio-Cam MRm, Carl Zeiss Micro-Imaging GmbH). Standard deviations were calculated from the average number of cells encapsulated per droplet on the patterned droplet array (see <strong><a href="#pone.0017455.s003">Fig. S3</a></strong> online).</p>


<h4 xpathlocation="/article[1]/body[1]/sec[3]/sec[3]/title[1]">Total RNA extraction and analysis</h4>
<p xpathlocation="/article[1]/body[1]/sec[3]/sec[3]/p[1]">To prevent contamination RNase AWAY&reg; (Sigma Aldrich), a ribonuclease decontaminant, was ejected through the valve, followed by nuclease-free water (Applied Biosystems) in preparation for cell encapsulating droplet ejection. Then, the entire area surrounding the ejector system was sprayed and wiped with RNaseZAP&reg;. Three rows in a 96-well plate were filled with 100 &micro;L of RLT Buffer solution (QIAGEN, 1% mercaptoethanol) to lyse the ejected cells and to preserve the RNA. The 96-well plate was then placed underneath the ejector and one droplet was ejected into each of the first nine wells of the first row of wells. This procedure was repeated with additional four rows of the plate, where two, three, four and five droplets were ejected in each well resulting to a total of five groups of droplet samples, each group having an increasing droplet volume and increasing number of cells ranging from ten to fifty cells. For consistency of bioanalysis, before the RNA analysis each row was divided into three triplicates. Hence, the analysis was based on average 30, 60, 90, 120, and 150 cells for each triplicate for bioanalysis. To investigate nonprinted cells, 0.5 &micro;l samples were drawn from the cell solution and pipetted into the three wells in each of the first three rows of the plate as controls. The controls had 1000 cells on average. Qiagen RNeasy Minikit (QIAGEN) was used to purify the total RNA. From the 96 well plate sample, triplicates were used to form single 300 &micro;l samples, after which each sample was transferred into a QIAshredder column in a 2 ml collection tube. The lysate was homogenized by placing the tubes in a microcentrifuge for 2 minutes at 5000 rpm. Each flow-through was transferred to a fresh 1.5 ml eppendorf tube and 300 &micro;l of 70% EtOH was added to each tube and mixed by pipetting. Each lysate solution was then applied to an RNeasy mini column in a 2 ml collection tube and spun for 15 seconds at 10,000&times;g, discarding the flow-through afterwards. A buffer, 350 &micro;L, RW1 (QIAGEN, RNeasy Minikit) was added to each column, and the columns were centrifuged for 15 seconds at 10,000&times;g, discarding the flow-through afterwards. This step was repeated. Then the columns were transferred to new 2 ml catch tubes. Next, 500 &micro;L of buffer RPE (QIAGEN, RNeasy Minikit) was pipetted into each RNeasy column, and the tubes were centrifuged for 15 seconds at 10,000&times;g (gravity), decanting the flow-through afterwards. Again, 500 &micro;L of RPE buffer was pipetted into the tubes, and the columns were centrifuged for 15 seconds at 10,000&times;g and placed in new 2 ml catch tubes and spun at full speed for one minute to ensure that the membranes were dry. The columns were then transferred to new 1.5 ml collection tubes. The tubes were left at room temperature for one minute after pipetting 52 &micro;L of ribonuclease-free water directly onto each membrane and then centrifuged for 1 minute at 10,000&times;g. The elution was repeated with the same volumes into the same tubes. The resulting eluant was speed vacuumed for 45 minutes at room temperature, and all samples were equilibrated to 15 &micro;L. With the 2100 Expert&reg; software, the isolated RNA was analyzed for quantity and quality using the Agilent 2100 Bioanalyzer (Agilent Technologies, Palo Alto, CA, using the RNA 6000 Pico Series II assay). All data is MIAME compliant and that the raw data has been deposited in a MIAME compliant database (GEO accession ID: GSE24330).</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/sec[3]/p[2]">The total RNA was amplified with Ambion's Illumina TotalPrep RNA amplification kit (Ambion, Inc. Austin, TX). A brief description of the amplification/purification process is as follows: Reverse transcription to synthesize first strand cDNA. 2&ndash;3 ng sample (total RNA) is primed with the T7 oligo (dT) primer to synthesize cDNA containing a T7 promoter sequence. Second strand cDNA synthesis converts the single-stranded cDNA into a double-stranded DNA (dsDNA) template for transcription. The reaction employs DNA polymerase and RNase H to simultaneously degrade the RNA and synthesize second strand cDNA. Next, cDNA purification removes RNA, primers, enzymes, and salts that would inhibit <em>in vitro</em> transcription. <em>In vitro</em> transcription to synthesize cRNA generates multiple copies of biotinylated cRNA from the double-stranded cDNA templates. This is the amplification and labeling step. Then cRNA purification removes unincorporated NTPs, salts, enzymes, and inorganic phosphate. After purification and fragmentation, the cRNA is ready to use with direct hybridization array kits. For hybridization to arrays (AppliedMicroarrays CodeLink Mouse Whole Genome Bioarray), 10 &micro;g of labeled cRNA was fragmented, denatured at 90&deg;C for 5 min, and then hybridized at 37&deg;C for 20 hours. The arrays were then washed in buffer at 47&deg;C for 60 minutes. After washing, the arrays were incubated at room temperature for 30 minutes with a fluorophore (Streptavidin-Alexa Fluor 647, Invitrogen Eugene, OR), run through a series of 5 minute rinses, spun dry, then scanned on a GenePix 4000B scanner (Axon Instrument, Sunnyvale, CA). The Mouse Whole Genome Bioarrays contains 38,313 single 30-mer oligonucleotide probes. For the gene expression measurements were normalized to a median expression level of 1.0 across the array. 1000 genes with the highest expression levels &ldquo;&gt;1&rdquo; in the experimental samples was used for comparisons. Three Bioarrays were run for both the experimental and control samples to enable statistical analysis of the reproducibility of the gene expression measurements for the most abundant genes.</p>


<h4 xpathlocation="/article[1]/body[1]/sec[3]/sec[4]/title[1]">Statistical modeling</h4>
<p xpathlocation="/article[1]/body[1]/sec[3]/sec[4]/p[1]">Statistics is widely adapted to biological analysis. Number of cells used in experiments are large, and these experiments follow the law of large numbers (LLN) and sampling processes are typically random, i.e. simple random sampling (SRS) <a href="#pone.0017455-Ross1">[33]</a>, <a href="#pone.0017455-Sa1">[34]</a>, <a href="#pone.0017455-Grinstead1">[35]</a>. In our statistical modeling, we described cell encapsulation as a random process with three random variables and their probability distribution functions (PDFs) where cells are encapsulated in droplets by a mechanical valve from a heterogeneous cell mixture. We followed three steps to model our target single cell encapsulation process. Based on experimental results, we (i) defined our system as a random system with random variables, (ii) checked whether each process was biased or un-biased, (iii) established statistical models with respect to each process and found parameters for the statistical models, e.g. mean (&mu;), variation (&sigma;), and Poisson distribution parameter (&lambda;), and (iv) evaluated overall system efficiency based on both CLT and SRS. Following these steps, we modeled our system as a random process (see <strong><a href="#pone.0017455.s006">Table S1</a> and <a href="#pone.0017455.s005">Fig. S5</a></strong> online).</p>

</div>

<div id="section4" xpathlocation="/article[1]/body[1]/sec[4]"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3 xpathlocation="noSelect">Supporting Information&nbsp;<a href="#top">Top</a></h3><a name="pone.0017455.s001" id="pone.0017455.s001"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0017455.s001">Figure S1. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[1]/caption[1]/p[1]"><strong>Homogeneous single cell droplet sorting system.</strong> A computerized xyz stage was synchronized with a pulse controller, Labview&trade;. The automated stage positioned the substrate with 5 &micro;m spatial resolution. Two ejectors and a 10&times; camera permitted <em>in-situ</em> imaging and lysing of the droplets. Schematic of droplet ejector showed cells flowing into the valve driven by a controlled air pressure pulse (34 kPa for 55 &micro;s). A heterogeneous sample, mixture of cells and media solution, was loaded into a reservoir. Each droplet was placed at a predetermined position.</p>
<p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[1]/caption[1]/p[2]">(TIF)</p>
<a name="pone.0017455.s002" id="pone.0017455.s002"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0017455.s002">Figure S2. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[2]/caption[1]/p[1]"><strong>Controlling droplet size.</strong> Image and illustration of ejected droplet in air were shown. (<strong>a</strong>) Stroboscopic images of ejected droplets. When the valve opening time and the gas pressure is not optimized, we observed satellite droplets. The sequential images were taken every second. (<strong>b</strong>) Stable droplets, D<sub>air</sub> = 245&plusmn;12 &micro;m (V&asymp;7.7&plusmn;1.2 nl), were generated during optimized ejection: <em>i.e.</em> 55 &micro;s valve opening time, 34.4 kPa pressurized nitrogen gas, and 25 Hz ejection frequency. The droplet size was measured at 2.3 mm distance from the ejector. (<strong>c</strong>) Droplet size at bottom surface. After landing on the surface, the diameter of the ejected droplet was determined by the surface tension and droplet volume. H<sub>droplet</sub> = 68&plusmn;2 &micro;m, D<sub>droplet</sub> = 525&plusmn;15 &micro;m, (V&asymp;7.6&plusmn;0.6 nl). Scale bars are 1 mm in <strong>a</strong> and <strong>b</strong>, 100 &micro;m in c, respectively.</p>
<p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[2]/caption[1]/p[2]">(TIF)</p>
<a name="pone.0017455.s003" id="pone.0017455.s003"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0017455.s003">Figure S3. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[3]/caption[1]/p[1]"><strong>High throughput droplet patterning system, images of printed droplet array.</strong> One hundred droplets were inspected for target homogeneous droplet sorting. (<strong>a&ndash;d</strong>) droplet array was generated under four different conditions, (<strong>a</strong>) 1&times;10<sup>5</sup> cells/ml with 10% target cell concentration, (<strong>b</strong>) 1&times;10<sup>5</sup> cells/ml with 50% target cell concentration, (<strong>c</strong>) 2&times;10<sup>5</sup> cells/ml with 10% target cell concentration, (<strong>d</strong>) 2&times;10<sup>5</sup> cells/ml with 50% target cell concentration. Scale bar, 1 mm.</p>
<p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[3]/caption[1]/p[2]">(TIF)</p>
<a name="pone.0017455.s004" id="pone.0017455.s004"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0017455.s004">Figure S4. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[4]/caption[1]/p[1]"><strong>Pre- and post-patterning cell viability.</strong> (<strong>a</strong>) Normalized cell viability was obtained comparing the two viabilities. Average and standard deviation of normalized viability were 96.4&plusmn;0.8%, 97.9&plusmn;2.3%, 97.2&plusmn;2.2%, and 96.0&plusmn;2.4% from 0.5&times;10<sup>5</sup> to 2&times;10<sup>5</sup> cells/ml of cell loading concentrations (n = 3 sets left y-axis reference). (<strong>b</strong>) Pre-patterning (flask) viabilities were measured based on 200 cells in a 10 &micro;l sample volume. Post-print cell viabilities were obtained from 2500 droplets for each cell loading concentration, <em>i.e.</em> 70 cells at 0.5&times;10<sup>5</sup> cells/ml and 575 cells at 2&times;10<sup>5</sup> cells/ml. Overall mean and standard deviation of pre- and post-printing cell viabilities were 96.7&plusmn;0.5% and 93.8&plusmn;1.1%, respectively (right hand y-axis).</p>
<p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[4]/caption[1]/p[2]">(TIF)</p>
<a name="pone.0017455.s005" id="pone.0017455.s005"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0017455.s005">Figure S5. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[5]/caption[1]/p[1]"><strong>Probability of encapsulating a single target cell in a droplet was presented by probability distribution functions, P(</strong><strong><em>X</em></strong><strong> = </strong><strong><em>X</em></strong><strong><sub>single target cell</sub>).</strong> (<strong>a</strong>) The number of homogeneous droplets was modeled using Poisson distribution in a random variable space, <em>i.e.</em> number of target cells. The model was verified using a coefficient, &lambda;, and experimental results. (<strong>b</strong>) According to four different cell loading concentrations and target cell concentrations, 16 Poisson distributions resulting from 4 cases of concentration times 4 cases of target cell mixture concentrations and their coefficients, &lambda;, were obtained comparing to experimental results. As cell loading density increases, target cell concentration, &lambda; values increase, &lambda;<sub>max</sub> = 0.95 and &lambda;<sub>min</sub> = 0.03. Based on the analysis, statistical models for 1.0&times;105 cells/ml with 10% target cell mixture were based on &lambda; = 0.05.</p>
<p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[5]/caption[1]/p[2]">(TIF)</p>
<a name="pone.0017455.s006" id="pone.0017455.s006"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0017455.s006">Table S1. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[6]/caption[1]/p[1]"><strong>Statistical modeling results for drop-on-demand target single cell encapsulation.</strong> (<strong>a</strong>) Randomness of process was verified by three variables, number of samples (n), tolerance (&epsilon;), and confidence level (1&minus;&alpha;) using an inequality<sup>(**)</sup>. Following the law of large numbers (LLN), minimum sample number was decided as 100 droplets for 90% confidence level and 15% tolerance. This sampling volume of a droplet represented (0.76 &micro;l = 10&times;10&times;7.6 nl) 0.76% of the total volume of the ejection reservoir (0.1 mL). (<strong>b</strong>) Random processes have different probability distribution functions in accordance with their parameters, <em>i.e.</em> number of cell containing droplets (<em>X<sub>drop</sub></em>), number of cells in a droplet (<em>X<sub>cell</sub></em>), number of target cells per droplet (<em>X<sub>target cell</sub></em>), and number of single target cell containing droplets (<em>X<sub>single target cell drop</sub></em>). These four different random variables are represented by three PDFs to statistically model the cell encapsulation process, i.e. binomial, Poisson, and normal distributions. We investigated probability values and parameters, &lambda;, for each case with respect to the cell loading concentrations, cell volume fraction, and percentage of target cells. (<strong>c</strong>) In the case of simple random sampling (SRS) process, statistical characteristics of a small sampling volume could represent the characteristics of a large population based on the central limit theorem (CLT). In our experiments, the target cell fraction (<em>F</em><sub>%</sub>) shows same concentration as the reservoir concentration for 10% to 50% at 1.0&times;10<sup>5</sup> cells/ml concentration (C<sub>opt</sub>) under conditions of 90% confidence level and 15% tolerance.</p>
<p xpathlocation="/article[1]/body[1]/sec[4]/supplementary-material[6]/caption[1]/p[2]">(DOCX)</p>
</div>





<div xpathlocation="noSelect"><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3 xpathlocation="noSelect">Acknowledgments&nbsp;<a href="#top">Top</a></h3>
<p xpathlocation="/article[1]/back[1]/ack[1]/p[1]">Microarray analyses were performed using BRB ArrayTools developed by Dr. Richard Simon and BRB-Array Tools Development Team. We would like to thank Pei-Ann Lin as a high school student, as well as Dr. Can Erdonmez and Dr. Emre Kayaalp for their kind feedback.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3 xpathlocation="noSelect">Author Contributions&nbsp;<a href="#top">Top</a></h3><p xpathlocation="noSelect"><span class="capture-id">Conceived and designed the experiments: SM Y-GK ML L-LH UD. Performed the experiments: SM ML. Analyzed the data: SM Y-GK LD ML EH RJ L-LH UD. Contributed reagents/materials/analysis tools: SM ML. Wrote the paper: SM Y-GK LD ML EH RJ L-LH UD.</span></p></div><div xpathlocation="noSelect"><a id="references" name="references" toc="references" title="References"></a><h3 xpathlocation="noSelect">References&nbsp;<a href="#top">Top</a></h3><ol class="references" xpathlocation="noSelect"><li xpathlocation="noSelect"><a name="pone.0017455-Marson1" id="pone.0017455-Marson1"></a><span class="authors">Marson A, Levine SS, Cole MF, Frampton GM, Brambrink T, et al. </span> (2008) Connecting microRNA Genes to the Core Transcriptional Regulatory Circuitry of Embryonic Stem Cells. Cell  134: 521&ndash;533. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Marson&title=Connecting%20microRNA%20Genes%20to%20the%20Core%20Transcriptional%20Regulatory%20Circuitry%20of%20Embryonic%20Stem%20Cells.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Bianco1" id="pone.0017455-Bianco1"></a><span class="authors">Bianco P, Robey PG</span> (2001) Stem cells in tissue engineering. Nature  414: 118&ndash;121. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Bianco&title=Stem%20cells%20in%20tissue%20engineering.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Takahashi1" id="pone.0017455-Takahashi1"></a><span class="authors">Takahashi K, Tanabe K, Ohnuki M, Narita M, Ichisaka T, et al. </span> (2007) Induction of Pluripotent Stem Cells from Adult Human Fibroblasts by Defined Factors. Cell  131: 861&ndash;872. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Takahashi&title=Induction%20of%20Pluripotent%20Stem%20Cells%20from%20Adult%20Human%20Fibroblasts%20by%20Defined%20Factors.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Zipori1" id="pone.0017455-Zipori1"></a><span class="authors">Zipori D</span> (2005) The Stem State: Plasticity Is Essential, Whereas Self-Renewal and Hierarchy Are Optional. Stem Cells  23: 719&ndash;726. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Zipori&title=The%20Stem%20State:%20Plasticity%20Is%20Essential,%20Whereas%20Self-Renewal%20and%20Hierarchy%20Are%20Optional.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Slack1" id="pone.0017455-Slack1"></a><span class="authors">Slack JMW</span> (2008) Origin of Stem Cells in Organogenesis. Science  322: 1498&ndash;1501. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Slack&title=Origin%20of%20Stem%20Cells%20in%20Organogenesis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Fan1" id="pone.0017455-Fan1"></a><span class="authors">Fan J-B, Chee MS, Gunderson KL</span> (2006) Highly parallel genomic assays. Nat Rev Genet  7: 632&ndash;644. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Fan&title=Highly%20parallel%20genomic%20assays.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Kurimoto1" id="pone.0017455-Kurimoto1"></a><span class="authors">Kurimoto K, Yabuta Y, Ohinata Y, Saitou M</span> (2007) Global single-cell cDNA amplification to provide a template for representative high-density oligonucleotide microarray analysis. Nat Protocols  2: 739&ndash;752. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Kurimoto&title=Global%20single-cell%20cDNA%20amplification%20to%20provide%20a%20template%20for%20representative%20high-density%20oligonucleotide%20microarray%20analysis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Svahn1" id="pone.0017455-Svahn1"></a><span class="authors">Svahn HA, Berg Avd</span> (2007) Single cells or large populations? Lab on a Chip  7: 544&ndash;546. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Svahn&title=Single%20cells%20or%20large%20populations?">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Fabbri1" id="pone.0017455-Fabbri1"></a><span class="authors">Fabbri R, Porcu E, Marsella T, Rocchetta G, Venturoli S, et al. </span> (2001) Human oocyte cryopreservation: new perspectives regarding oocyte survival. Hum Reprod  16: 411&ndash;416. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Fabbri&title=Human%20oocyte%20cryopreservation:%20new%20perspectives%20regarding%20oocyte%20survival.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Leong1" id="pone.0017455-Leong1"></a><span class="authors">Leong KG, Wang B-E, Johnson L, Gao W-Q</span> (2008) Generation of a prostate from a single adult stem cell. Nature  456: 804&ndash;808. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Leong&title=Generation%20of%20a%20prostate%20from%20a%20single%20adult%20stem%20cell.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Walker1" id="pone.0017455-Walker1"></a><span class="authors">Walker A, Parkhill J</span> (2008) Single-cell genomics. Nat Rev Micro  6: 176&ndash;177. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Walker&title=Single-cell%20genomics.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Sun1" id="pone.0017455-Sun1"></a><span class="authors">Sun W, Darling A, Starly B, Nam J</span> (2004) Computer-aided tissue engineering: overview, scope and challenges. Biotechnol Appl Biochem  39: 29&ndash;47. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Sun&title=Computer-aided%20tissue%20engineering:%20overview,%20scope%20and%20challenges.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Mironov1" id="pone.0017455-Mironov1"></a><span class="authors">Mironov V, Visconti RP, Kasyanov V, Forgacs G, Drake CJ, et al. </span> (2009) Organ printing: tissue spheroids as building blocks. Biomaterials  30: 2164&ndash;2174. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Mironov&title=Organ%20printing:%20tissue%20spheroids%20as%20building%20blocks.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Baret1" id="pone.0017455-Baret1"></a><span class="authors">Baret JC, Miller OJ, Taly V, Ryckelynck M, El-Harrak A, et al. </span> (2009) Fluorescence-activated droplet sorting (FADS): efficient microfluidic cell sorting based on enzymatic activity. Lab Chip  9: 1850&ndash;1858. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Baret&title=Fluorescence-activated%20droplet%20sorting%20%28FADS%29:%20efficient%20microfluidic%20cell%20sorting%20based%20on%20enzymatic%20activity.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Song1" id="pone.0017455-Song1"></a><span class="authors">Song YS, Adler D, Xu F, Kayaalp E, Nureddin A, et al. </span> (2010) Vitrification and levitation of a liquid droplet on liquid nitrogen. Proc Natl Acad Sci U S A  107: 4596&ndash;4600. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Song&title=Vitrification%20and%20levitation%20of%20a%20liquid%20droplet%20on%20liquid%20nitrogen.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Kvist1" id="pone.0017455-Kvist1"></a><span class="authors">Kvist T, Ahring BK, Lasken RS, Westermann P</span> (2007) Specific single-cell isolation and genomic amplification of uncultured microorganisms. Appl Microbiol Biotechnol  74: 926&ndash;935. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Kvist&title=Specific%20single-cell%20isolation%20and%20genomic%20amplification%20of%20uncultured%20microorganisms.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Zhang1" id="pone.0017455-Zhang1"></a><span class="authors">Zhang K, Martiny AC, Reppas NB, Barry KW, Malek J, et al. </span> (2006) Sequencing genomes from single cells by polymerase cloning. Nat Biotechnol  24: 680&ndash;686. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Zhang&title=Sequencing%20genomes%20from%20single%20cells%20by%20polymerase%20cloning.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Geigl1" id="pone.0017455-Geigl1"></a><span class="authors">Geigl JB, Speicher MR</span> (2007) Single-cell isolation from cell suspensions and whole genome amplification from single cells to provide templates for CGH analysis. Nat Protocols  2: 3173&ndash;3184. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Geigl&title=Single-cell%20isolation%20from%20cell%20suspensions%20and%20whole%20genome%20amplification%20from%20single%20cells%20to%20provide%20templates%20for%20CGH%20analysis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Raghunathan1" id="pone.0017455-Raghunathan1"></a><span class="authors">Raghunathan A, Ferguson HR Jr, Bornarth CJ, Song W, Driscoll M, et al. </span> (2005) Genomic DNA amplification from a single bacterium. Appl Environ Microbiol  71: 3342&ndash;3347. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Raghunathan&title=Genomic%20DNA%20amplification%20from%20a%20single%20bacterium.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Farragher1" id="pone.0017455-Farragher1"></a><span class="authors">Farragher S, Tanney A, Kennedy R, Paul Harkin D</span> (2008) RNA expression analysis from formalin fixed paraffin embedded tissues. Histochemistry and Cell Biology  130: 435&ndash;445. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Farragher&title=RNA%20expression%20analysis%20from%20formalin%20fixed%20paraffin%20embedded%20tissues.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Zeng1" id="pone.0017455-Zeng1"></a><span class="authors">Zeng Y, Novak R, Shuga J, Smith MT, Mathies RA</span> (2010) High-performance single cell genetic analysis using microfluidic emulsion generator arrays. Anal Chem  82: 3183&ndash;3190. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Zeng&title=High-performance%20single%20cell%20genetic%20analysis%20using%20microfluidic%20emulsion%20generator%20arrays.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-DiCarlo1" id="pone.0017455-DiCarlo1"></a><span class="authors">Di Carlo D, Wu LY, Lee LP</span> (2006) Dynamic single cell culture array. Lab Chip  6: 1445&ndash;1449. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Di%20Carlo&title=Dynamic%20single%20cell%20culture%20array.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Ino1" id="pone.0017455-Ino1"></a><span class="authors">Ino K, Okochi M, Konishi N, Nakatochi M, Imai R, et al. </span> (2008) Cell culture arrays using magnetic force-based cell patterning for dynamic single cell analysis. Lab on a Chip  8: 134&ndash;142. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Ino&title=Cell%20culture%20arrays%20using%20magnetic%20force-based%20cell%20patterning%20for%20dynamic%20single%20cell%20analysis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-White1" id="pone.0017455-White1"></a><span class="authors">White RA 3rd, Blainey PC, Fan HC, Quake SR</span> (2009) Digital PCR provides sensitive and absolute calibration for high throughput sequencing. BMC Genomics  10: 116. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=White&title=Digital%20PCR%20provides%20sensitive%20and%20absolute%20calibration%20for%20high%20throughput%20sequencing.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Ottesen1" id="pone.0017455-Ottesen1"></a><span class="authors">Ottesen EA, Hong JW, Quake SR, Leadbetter JR</span> (2006) Microfluidic Digital PCR Enables Multigene Analysis of Individual Environmental Bacteria. Science  314: 1464&ndash;1467. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Ottesen&title=Microfluidic%20Digital%20PCR%20Enables%20Multigene%20Analysis%20of%20Individual%20Environmental%20Bacteria.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Toriello1" id="pone.0017455-Toriello1"></a><span class="authors">Toriello NM, Douglas ES, Thaitrong N, Hsiao SC, Francis MB, et al. </span> (2008) Integrated microfluidic bioprocessor for single-cell gene expression analysis. Proceedings of the National Academy of Sciences  105: 20173&ndash;20178. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Toriello&title=Integrated%20microfluidic%20bioprocessor%20for%20single-cell%20gene%20expression%20analysis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Love1" id="pone.0017455-Love1"></a><span class="authors">Love JC, Ronan JL, Grotenbreg GM, van der Veen AG, Ploegh HL</span> (2006) A microengraving method for rapid selection of single cells producing antigen-specific antibodies. Nat Biotech  24: 703&ndash;707. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Love&title=A%20microengraving%20method%20for%20rapid%20selection%20of%20single%20cells%20producing%20antigen-specific%20antibodies.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Lin1" id="pone.0017455-Lin1"></a><span class="authors">Lin LI, Chao SH, Meldrum DR</span> (2009) Practical, microfabrication-free device for single-cell isolation. PLoS One  4: e6710. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Lin&title=Practical,%20microfabrication-free%20device%20for%20single-cell%20isolation.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Fu1" id="pone.0017455-Fu1"></a><span class="authors">Fu AY, Spence C, Scherer A, Arnold FH, Quake SR</span> (1999) A microfabricated fluorescence-activated cell sorter. Nat Biotech  17: 1109&ndash;1111. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Fu&title=A%20microfabricated%20fluorescence-activated%20cell%20sorter.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Hong1" id="pone.0017455-Hong1"></a><span class="authors">Hong JW, Studer V, Hang G, Anderson WF, Quake SR</span> (2004) A nanoliter-scale nucleic acid processor with parallel architecture. Nat Biotech  22: 435&ndash;439. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Hong&title=A%20nanoliter-scale%20nucleic%20acid%20processor%20with%20parallel%20architecture.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Pushkarev1" id="pone.0017455-Pushkarev1"></a><span class="authors">Pushkarev D, Neff NF, Quake SR</span> (2009) Single-molecule sequencing of an individual human genome. Nat Biotech  27: 847&ndash;850. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Pushkarev&title=Single-molecule%20sequencing%20of%20an%20individual%20human%20genome.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Marcus1" id="pone.0017455-Marcus1"></a><span class="authors">Marcus JS, Anderson WF, Quake SR</span> (2006) Microfluidic single-cell mRNA isolation and analysis. Anal Chem  78: 3084&ndash;3089. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Marcus&title=Microfluidic%20single-cell%20mRNA%20isolation%20and%20analysis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Ross1" id="pone.0017455-Ross1"></a><span class="authors">Ross SM</span> (2008) A first course in probability. Harlow: Pearson Education.  </li><li xpathlocation="noSelect"><a name="pone.0017455-Sa1" id="pone.0017455-Sa1"></a><span class="authors">Sa M</span> (2007) Applied statistics using SPSS, STATISTICA, MATLAB and R. Berlin; New York: Springer.  </li><li xpathlocation="noSelect"><a name="pone.0017455-Grinstead1" id="pone.0017455-Grinstead1"></a><span class="authors">Grinstead CM, Snell JL</span> (1997) Introduction to probability. Providence, RI: American Mathematical Society.  </li><li xpathlocation="noSelect"><a name="pone.0017455-Moon1" id="pone.0017455-Moon1"></a><span class="authors">Moon S, Keles HO, Ozcan A, Khademhosseini A, Haegstrom E, et al. </span> (2009) Integrating microfluidics and lensless imaging for point-of-care testing. Biosensors and Bioelectronics  24: 3208&ndash;3214. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Moon&title=Integrating%20microfluidics%20and%20lensless%20imaging%20for%20point-of-care%20testing.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Megason1" id="pone.0017455-Megason1"></a><span class="authors">Megason SG, Fraser SE</span> (2007) Imaging in systems biology. Cell  130: 784&ndash;795. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Megason&title=Imaging%20in%20systems%20biology.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Hwang1" id="pone.0017455-Hwang1"></a><span class="authors">Hwang YS, Chung BG, Ortmann D, Hattori N, Moeller HC, et al. </span> (2009) Microwell-mediated control of embryoid body size regulates embryonic stem cell fate via differential expression of WNT5a and WNT11. Proc Natl Acad Sci U S A  106: 16978&ndash;16983. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Hwang&title=Microwell-mediated%20control%20of%20embryoid%20body%20size%20regulates%20embryonic%20stem%20cell%20fate%20via%20differential%20expression%20of%20WNT5a%20and%20WNT11.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Fu2" id="pone.0017455-Fu2"></a><span class="authors">Fu AY, Chou HP, Spence C, Arnold FH, Quake SR</span> (2002) An Integrated Microfabricated Cell Sorter. Analytical Chemistry  74: 2451&ndash;2457. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Fu&title=An%20Integrated%20Microfabricated%20Cell%20Sorter.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0017455-Marcy1" id="pone.0017455-Marcy1"></a><span class="authors">Marcy Y, Ishoey T, Lasken RS, Stockwell TB, Walenz BP, et al. </span> (2007) Nanoliter reactors improve multiple displacement amplification of genomes from single cells. PLoS Genet  3: 1702&ndash;1708. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Marcy&title=Nanoliter%20reactors%20improve%20multiple%20displacement%20amplification%20of%20genomes%20from%20single%20cells.">

                    Find this article online
                  </a></li></ol></div>

</div>

</div>
<div style="display: none;">
<div dojotype="ambra.widget.RegionalDialog" id="AnnotationDialog" style="padding: 0pt; margin: 0pt;">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Post Your Note (For Public Viewing)</span></h5>
      <div class="posting pane">
        <form name="createAnnotation" id="createAnnotation" method="post" action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455">
          <input name="target" value="info:doi/10.1371/journal.pone.0017455" type="hidden">
          <input name="startPath" value="" type="hidden">
          <input name="startOffset" value="" type="hidden">
          <input name="endPath" value="" type="hidden">
          <input name="endOffset" value="" type="hidden">
          <input name="commentTitle" id="commentTitle" value="" type="hidden">
          <input name="comment" id="commentArea" value="" type="hidden">
          <input name="ciStatement" id="statementArea" value="" type="hidden">
          <input name="isCompetingInterest" id="isCompetingInterest" value="false" type="hidden">
          <input name="noteType" id="noteType" value="" type="hidden">
          <fieldset>
            <legend>Compose Your Note</legend>
            <span id="submitMsg" class="error" style="display: none;"></span>
            <table class="layout">
              <tbody><tr>
                <td>
                  <label for="cNoteType">This is a </label><select name="cNoteType" id="cNoteType"><option value="note">note</option><option value="correction">correction</option></select>
                  <span id="cdls" style="visibility: hidden; margin-left: 0.3em; white-space: nowrap;"><a href="http://www.plosone.org/static/commentGuidelines.action?target=info%3Adoi%2F10.1371%2Fjournal.pone.0017455#corrections">What are corrections?</a></span>
                  <label for="cTitle"><span class="none">Enter your note title</span><!-- error message text <em>A title is required for all public notes</em>--></label>
                  <input name="cTitle" id="cTitle" value="Enter your note title..." class="title" alt="Enter your note title..." type="text">
                  <label for="cArea"><span class="none">Enter your note</span><!-- error message text <em>Please enter your note</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your note..." alt="Enter your note...">Enter your note...</textarea>
                  <input name="isPublic" value="true" type="hidden">
                </td>
                <td>&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" checked="checked" name="competingInterest" value="false" type="radio"> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" name="competingInterest" value="true" type="radio"> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled="disabled" value="Enter your competing interests..." alt="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td colspan="3" class="buttons">
                  <input value="Cancel" title="Click to close and cancel" id="btn_cancel" type="button">
                  <input value="Submit" title="Click to post your note publicly" id="btn_post" class="primary" type="button">
                </td>
              </tr>
            </tbody></table>
          </fieldset>
        </form>
      </div>
    </div>
    <div class="tip" id="dTip"></div>
  </div>
</div><div dojotype="ambra.widget.ContextAction" id="ContextActionDialog" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="caTipu"></div>
    <div class="contextActionContent">
      <h5><img src="tooltip_addannotation.gif"> Add a note to this text.</h5>
      Please follow our <a href="http://www.plosone.org/static/commentGuidelines.action">guidelines for notes and comments</a> and review our <a href="http://www.plosone.org/static/competing.action">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
      <ul>
        <li>Remarks that could be interpreted as allegations of misconduct</li>
        <li>Unsupported assertions or statements</li>
        <li>Inflammatory or insulting language</li>
      </ul>
      <form name="contextActionForm" id="contextActionForm" class="clearfix buttons" method="post" action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455">
        <input name="Continue" value="Continue" id="ContextActionDialogContinueButton" onmouseup="ambra.displayAnnotationContext.startComment(event);" title="Add a note to this text" class="primary" type="button">
        <input name="Cancel" value="Cancel" id="ContextActionDialogCancelButton" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);" title="Close this Window" type="button">
      </form>
    </div>
    <div class="tip" id="caTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.ContextAction" id="ContextActionDialogNotLogged" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canlTipu"></div>
    <div class="contextActionContent">
      <h5><img src="tooltip_addannotation.gif"> Add a note to this text.</h5>
      You must be logged in to add a note to an article.
      You may log in by <a onmousedown="ambra.displayAnnotationContext.disconnect(event);" href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2Finfo%3Adoi%2F10.1371%2Fjournal.pone.0017455">clicking here</a> or <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">cancel this note</a>.
    </div>
    <div class="tip" id="canlTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.ContextAction" id="ContextActionDialogBadSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canBDTipu"></div>
    <div class="contextActionContent">
      <h5 class="annotation icon"><img src="tooltip_addannotation.gif"> Add a note to this text.</h5>
      You cannot annotate this area of the document. <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canBDTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.ContextAction" id="ContextActionDialogBadRangeSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canbrTipu"></div>
    <div class="contextActionContent">
      <h5><img src="tooltip_addannotation.gif"> Add a note to this text.</h5>
      You cannot create an annotation that spans different sections of the document; please adjust your selection.<br>
      <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canbrTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.RegionalDialog" id="CommentDialog" style="padding: 0pt; margin: 0pt;">
  <div class="dialog preview">
    <div class="tipu" id="cTipu"></div>
    <div class="btn close" id="btn_close" title="Click to close"><a title="Click to close">Close</a></div>
    <div id="cmtContainer" class="comment">
      <h6 id="viewCmtTitle"></h6>
      <div class="detail" id="viewCmtDetail"></div>
      <div class="contentwrap" id="viewComment"></div>
      <div class="contentwrap" id="viewCIStatement"></div>
      <div class="detail" id="viewLink">
        <!--<a href="#" class="commentary icon" title="Click to view full thread and respond">View all responses</a>
        <a href="#" class="respond tooltip" title="Click to respond to this posting">Respond to this</a>-->
      </div>
    </div>
    <div class="tip" id="cTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.RegionalDialog" id="CommentDialogMultiple" style="padding: 0pt; margin: 0pt;">
  <div class="dialog multiple preview">
    <div class="tipu" id="mTipu"></div>
    <div class="btn close" id="btn_close_multi" title="Click to close"><a title="Click to close">Close</a></div>
    <ol id="multilist"></ol>
    <br>
    <div id="multidetail"></div>
    <div class="tip" id="mTip"></div>
  </div>
</div>


<div dojotype="dijit.Dialog" id="Rating">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Rate This Article</span></h5>
      <div class="instructions">Please follow our <a href="http://www.plosone.org/static/ratingGuidelines.action">guidelines for rating</a> and review our <a href="http://www.plosone.org/static/competing.action">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
        <ol>
          <li>Remarks that could be interpreted as allegations of misconduct</li>
          <li>Unsupported assertions or statements</li>
          <li>Inflammatory or insulting language</li>
        </ol>
      </div>
      <div class="posting pane">
        <form name="ratingForm" id="ratingForm" method="post" action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0017455">
          <input name="articleURI" value="info:doi/10.1371/journal.pone.0017455" type="hidden">  
          <input name="commentTitle" id="commentTitle" value="" type="hidden">
          <input name="comment" id="commentArea" value="" type="hidden">
          <input name="ciStatement" id="statementArea" value="" type="hidden">
          <input name="isCompetingInterest" id="isCompetingInterest" value="" type="hidden">
          <fieldset>
            <legend>Compose Your Annotation</legend>
            <span id="submitRatingMsg" class="error" style="display: none;"></span>
            <table class="layout">
              <tbody><tr>
                <td rowspan="2">
                  <label for="insight">Insight</label>
                  <ul class="star-rating rating edit" title="Rate insight" id="rateInsight">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Bland" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'insight', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Profound" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 5);">5</a></li>
                  </ul>    
                  <input name="insight" title="insight" value="" type="hidden">
                  <label for="reliability">Reliability</label>
                  <ul class="star-rating rating edit" title="Rate reliability" id="rateReliability">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Tenuous" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'reliability', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Unassailable" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 5);">5</a></li>
                  </ul>    
                  <input name="reliability" title="reliability" value="" type="hidden">
                  <label for="style">Style</label>
                  <ul class="star-rating rating edit" title="Rate style" id="rateStyle">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Crude" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'style', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Elegant" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 5);">5</a></li>
                  </ul>    
                  <input name="style" title="style" value="" type="hidden">
                  <label for="cTitle"><span class="none">Enter your comment title</span><!-- error message text <em>A title is required for all public annotations</em>--></label>
                  <input name="cTitle" id="cTitle" value="Enter your comment title..." class="title" alt="Enter your comment title..." type="text">
                  <label for="cArea"><span class="none">Enter your comment</span><!-- error message text <em>Please enter your annotation</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your comment..." alt="Enter your comment...">Enter your comment...</textarea>
                </td>
                <td rowspan="2">&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" name="competingInterest" value="false" type="radio"> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" name="competingInterest" value="true" type="radio"> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled="disabled" value="Enter your competing interests..." title="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td class="buttons">
                  <input value="Cancel" title="Click to close and cancel" id="btn_cancel_rating" type="button">
                  <input value="Submit" title="Click to post your annotation publicly" id="btn_post_rating" class="primary" type="button">
                </td>
              </tr>
            </tbody></table>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
</div>
<div dojotype="ambra.widget.LoadingCycle" id="LoadingCycle" class="loadingCycler">
  <img src="loading.gif" title="Loading..." height="58" width="58">
</div>
</div>
</div><!-- end : main contents -->

</div>
<!-- end : container -->	

<!-- begin : footer -->
<div id="ftr">
  <p><span>All site content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License 2.5" tabindex="200">Creative Commons Attribution License</a>.</span></p>
  <ul>
    <li><a href="http://www.plosone.org/static/privacy.action" title="PLoS Privacy Statement" tabindex="501">Privacy Statement</a></li>
    <li><a href="http://www.plosone.org/static/terms.action" title="PLoS Terms of Use" tabindex="502">Terms of Use</a></li>
    <li><a href="http://www.plos.org/advertise/" title="Advertise With PLoS" tabindex="503">Advertise</a></li>
    <li><a href="http://www.plos.org/journals/embargopolicy.html" title="PLoS Embargo Policy" tabindex="504">Media Inquiries</a></li>
    <li><a href="http://www.plos.org/journals/print.html" title="PLoS in Print" tabindex="505">PLoS in Print</a></li>
    <li><a href="http://www.plosone.org/static/sitemap.action" title="Site Map" tabindex="506">Site Map</a></li>
    <li><a href="http://www.plos.org/" title="PLoS.org" tabindex="507">PLoS.org</a></li>
  </ul>
  <div class="powered">
  <ul>
    <li><a href="http://www.plosone.org/static/releaseNotes.action" title="Ambra | Release Notes">Ambra 1.1 </a></li>
    <li>Managed Colocation provided by <a href="http://www.isc.org/" title="Internet Systems Consortium">Internet Systems Consortium</a>.</li>
  </ul>
  </div>
</div>
<!-- end : footer -->
<script type="text/javascript">

  var _namespace="";
    var loggedIn = false;

  var almHost = "http://alm.plos.org";
  var pubGetHost = "http://pubget.com/widgetizer/links_hash";
  var appContext = "";

  // Safari v3.1.1 "console.debug" issue (http://trac.dojotoolkit.org/ticket/6849) workaround
  if (/3[\.0-9]+ Safari/.test(navigator.appVersion)) {
    window.console = {
      origConsole: window.console,
      log: function(s){ this.origConsole.log(s); },
      info: function(s){ this.origConsole.info(s); },
      error: function(s){ this.origConsole.error(s); },
      warn: function(s){ this.origConsole.warn(s); }
    };
  }

  var djConfig = {
    // don't debug for IE - as dojo's firebug lite module is error prone in IE
    isDebug: false,
    parseOnLoad: true
  };
</script>
<script type="text/javascript" src="dojo.js"></script>
<script type="text/javascript" src="ambra.js"></script>
<script type="text/javascript" src="init_global.js"></script>
<script type="text/javascript" src="init_article.js"></script>
<script type="text/javascript" src="init_ratings.js"></script>
<script type="text/javascript" src="init_article_body.js"></script>
<script type="text/javascript" src="init_article_rhc.js"></script>
<script type="text/javascript" src="alm.js"></script>
<script type="text/javascript" src="articleviewscumulative.js"></script>

<script type="text/javascript">

var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">

var pageTracker = _gat._getTracker("UA-338393-1");
pageTracker._trackPageview();
pageTracker._setDomainName("www.plosone.org");
</script>

<script type="text/javascript">

var _sf_async_config={uid:16579,domain:"plosone.org"};
(function(){
  function loadChartbeat() {
    window._sf_endpt=(new Date()).getTime();
    var e = document.createElement('script');
    e.setAttribute('language', 'javascript');
    e.setAttribute('type', 'text/javascript');
    e.setAttribute('src',
       (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
       "js/chartbeat.js");
    document.body.appendChild(e);
  }
  var oldonload = window.onload;
  window.onload = (typeof window.onload != 'function') ?
     loadChartbeat : function() { oldonload(); loadChartbeat(); };
})();

</script>

</body>
</html>
